# Structural visualization of the tubulin folding pathway directed by human chaperonin TRiC/CCT # Graphical abstract # Authors Daniel Gestaut, Yanyan Zhao, Junsun Park, ..., Soung-Hun Roh, Wah Chiu, Judith Frydman # Correspondence shroh@snu.ac.kr (S.-H.R.), wahc@stanford.edu (W.C.), jfrydman@stanford.edu (J.F.) # In brief Starting from unstructured tubulin bound to the chaperone prefoldin, the steps of TriC-guided tubulin folding are revealed in structural detail illuminating the folding mechanism.  # Highlights -Reconstitution of chaperone-mediated tubulin folding with human prefoldin and TRiC/CCT-Prefoldin delivers unstructured tubulin directly into the TRiC chamber-TRiC orchestrates ATP-driven tubulin folding via progressively formed intermediates-Specific contacts within the TRiC chamber shape the tubulin folding landscape # Article # Structural visualization of the tubulin folding pathway directed by human chaperonin TRiC/CCT Daniel Gestaut, $^{1,7}$ Yanyan Zhao, $^{1,7}$ Junsun Park, $^{5}$ Boxue Ma, $^{3}$ Alexander Leitner, $^{4}$ Miranda Collier, $^{1}$ Grigore Pintilie, $^{3}$ Soung-Hun Roh, $^{5,\ast}$ Wah Chiu, $^{3,6,\ast}$ and Judith Frydman $^{1,2,8,\ast}$ $^{1}$ Department of Biology, Stanford University, Stanford, CA, USA $^{2}$ Department of Genetics, Stanford University, Stanford, CA, USA $^{3}$ Department of Bioengineering, Stanford University, Stanford, CA, USA $^{4}$ Institute of Molecular Systems Biology, Department of Biology, ETH Zurich, 8 Zurich, Switzerland $^{5}$ School of Biological Sciences, Institute of Molecular Biology and Genetics, Seoul National University, Seoul, South Korea $^{6}$ SSRL, SLAC National Accelerator Laboratory, Menlo Park, CA, USA $^{7}$ These authors contributed equally $^{8}$ Lead contact $^{\ast}$ Correspondence: shroh@snu.ac.kr (S.-H.R.), wahc@stanford.edu (W.C.), jfrydman@stanford.edu (J.F.) https://doi.org/1.1/j.cell.2.1.0 # SUMMARY The ATP-dependent ring-shaped chaperonin TRiC/CCT is essential for cellular proteostasis. To uncover why some eukaryotic proteins can only fold with TRiC assistance, we reconstituted the folding of $\beta$ -tubulin using human prefoldin and TRiC. We find unstructured $\beta$ -tubulin is delivered by prefoldin to the open TRiC chamber followed by ATP-dependent chamber closure. Cryo-EM resolves four near-atomic-resolution structures containing progressively folded $\beta$ -tubulin intermediates within the closed TRiC chamber, culminating in native tubulin. This substrate folding pathway appears closely guided by site-specific interactions with conserved regions in the TRiC chamber. Initial electrostatic interactions between the TRiC interior wall and both the folded tubulin N domain and its C-terminal E-hook tail establish the native substrate topology, thus enabling C-domain folding. Intrinsically disordered CCT C termini within the chamber promote subsequent folding of tubulin's core and middle domains and GTP-binding. Thus, TRiC's chamber provides chemical and topological directives that shape the folding landscape of its obligate substrates. # INTRODUCTION How proteins fold in vivo is a central question in biology, highly relevant to human health, as many diseases ranging from cancer to neurodegeneration are linked to protein misfolding.1-5 Several classes of molecular chaperones play central roles in the folding of newly translated polypeptides.6-8 While some chaperones, such as NAC or Hsp facilitate folding of a wide range of proteins, others, such as the ring-shaped chaperonin TRiC/CCT facilitate folding of a subset of proteins comprising $\sim\%$ of the eukaryotic proteome.9 The proteins that require TRiC for folding, many of them essential for viability, are distinguished by their aggregation propensity and topological complexity.9,1 In vivo, TRiC functions in cooperation with upstream chaperones, including Hsp and prefoldin (PFD, also called GIMc). Prefoldin binds directly to TRiC through an electrostatic pivot, suggesting a direct mechanism of substrate transfer.1 How TRiC/CCT assists folding is intriguing because several eukaryotic proteins can only fold with the assistance of TRiC. These proteins cannot fold either spontaneously nor with any other chaperone, under any known condition.1,1 An explanation for this obligate requirement is lacking. An intriguing hypoth esis is that the information present in their amino acid sequence does not suffice to reach the native state. In this hypothesis, TRiC may contribute the chemical and topological environment necessary to guide their folding, suggesting a co-evolution between TRiC and some eukaryotic proteins. TRiC is a $\sim$ MDa chaperonin, consisting of two identical rings stacked back to back.1 Unlike bacterial or archaeal chaperonins, each TRiC ring is assembled from eight different subunits, called CCT-CCT, in an evolutionarily conserved arrangement.1 Access to the central chamber of each ring is regulated in an ATP-dependent manner by a built-in lid, formed by flexible loops protruding from each subunit. The lid is open in the unliganded apo-state of TRiC, allowing substrates to bind through subunit-specific contacts in the apical domains.1,1 ATP hydrolysis leads to lid closure, encapsulating the substrate within the central chamber, where folding occurs.1,1 The hetero-oligomeric nature of TRiC is likely key to its unique ability to fold obligate substrates. Subunit divergence likely contributes to the folding process in various ways. The specific subunit arrangement segregates substrate and ATP binding properties within the ring and produces an asymmetric charge  Figure. Reconstitution of the tubulin folding pathway from purified components (A) Prefoldin (PFD) and TRiC-mediated tubulin folding pathway reconstituted from purified components. (B) N-PAGE analysis of $\beta$ tubulin interactions with prefoldin and TRiC. Addition of nucleotide (ATP/GTP) drives folding and partial binding of $\beta$ tubulin to TBCA. (C) Proteinase K sensitivity assay (PK) of tubulin conformation along the prefoldin-TRiC folding pathway. (D) Tubulin conformation along the chaperone folding pathway was further analyzed as indicated. (E) PK assay of $\beta$ tubulin copurified with either TRiC or group II archaeal chaperonin CPN with or without ATP/GTP addition. See also Figure S. distribution within the closed chamber.1,1 Unfolded substrates such as actin, AML, or VHL bind open TRiC through subunit-specific contacts with CCT apical domains.1,1-2 This could bias the initial topology of the TRiC-bound substrate to a conformation that is productive for folding. Analysis of TRiC-mediated actin folding indicates that the gradient of ATP affinities within the ring promotes stepwise release during chamber closure to facilitate folding.1,1 However, how folding occurs inside the closed TRiC folding chamber is not understood. Is the bound substrate released into the ATP-closed chaperonin chamber, as shown for bacterial chaperonin GroEL-ES? How do the properties of the TRiC chamber affect the folding pathway of its substrates? To address these questions, we reconstituted and analyzed the prefoldin-TRiC-mediated folding pathway of the archetype TRiC obligate substrates, $\alpha$ and $\beta$ -tubulin. Prefoldin maintains tubulin in a highly unstructured state and directly transfers substrate into the TRiC chamber. ATP addition then leads to lid closure and tubulin encapsulation and folding within the closed TRiC chamber. Biochemical and cryoelectron microscopy cryo-EM) structural analyses of the chaperone-mediated folding reaction reveal highly specific substrate contacts with the inner chamber of TRiC. These electrostatic and polar tubulin contacts with individual CCT subunits direct the tubulin folding trajectory by promoting domain-wise formation of four progressively folded intermediates culminating in the native state. # RESULTS # Reconstituting the human prefoldin-TRiC pathway for efficient tubulin folding $\alpha$ or $\beta$ -tubulin monomers are folded in vivo by a chaperone pathway consisting of prefoldin and TRiC/CCT,2 and subsequently steered into tubulin dimers by a complex network of poorly understood assembly factors.2,2 We reconstituted the prefoldin-TRiC-tubulin monomer folding pathway using purified human components (Figures 1A and S1A, B). Folding reactions were performed starting from prefoldin-bound $\alpha$ and $\beta$ -tubulin complexes (Figure S1C) that were incubated with purified human TRiC in the presence and absence of ATP. Folding reactions were analyzed using native PAGE (N-PAGE) to assess formation of tubulin chaperone complexes during the folding reaction while limited-proteolysis assays followed by gel electrophoresis (LIP-PAGE) were used to examine the conformation of chaperone-bound tubulin (Figures 1B and 1C). N-PAGE revealed that TRiC addition to prefoldin: $\alpha$ -tubulin or prefoldin: $\beta$ -tubulin complexes led to formation of a ternary complex with TRiC, reflected in a characteristic mobility shift  for $\beta$ -tubulin and S1D for $\alpha$ -tubulin).1 Subsequent ATP addition led to production of folded protease-resistant tubulin monomer ; note GTP is added to stabilize folded tubulin). To further assess whether TRiC promotes ATP-dependent tubulin folding, we incubated these reactions with two factors known to bind to either $\alpha$ or $\beta$ -tubulin monomers, called TBCB and TBCA,  respectively.2-2 N-PAGE revealed that, in the presence of ATP, TRiC yielded tubulin competent to form a characteristic complex with these monomer-binding factors  and S1D). respectively. $^{2 -2}$ N-PAGE revealed that, in the presence of ATP, TRiC yielded tubulin competent to form a characteristic complex with these monomer-binding factors  and S1D).Susceptibility to low concentrations of non-specific protease proteinase K (PK) is a hallmark of unstructured polypeptides, while folded domains tend to exhibit enhanced PK resistance. $^{2,2}$ We used LIP-PAGE with PK to characterize the conformation of $\alpha$ -and $\beta$ -tubulin folding intermediates along the prefoldin:TRiC chaperone pathway (LIP-PAGE, Figures 1C and S1E). Similar results were obtained for $\alpha$ -and $\beta$ -tubulins, suggesting a shared mechanism of chaperone action for these proteins (Figures S1D and S1E). As expected, folded tubulin in $\alpha /\beta$ dimers is nearly fully PK resistant, except for some degradation of flexible C-terminal tails. In contrast, both $\alpha$ -and $\beta$ -tubulin bound to prefoldin were highly PK sensitive, indicating the bound substrate adopts an unstructured protease-sensitive conformation. Addition of TRiC and ensuing formation of a ternary prefoldin:tubulin:TRiC complex led to enhanced $\alpha$ -and $\beta$ -tubulin protection against PK, suggesting tubulin becomes compacted upon binding to TRiC. We reached similar conclusions using an antibody directed against the N-terminal domain of tubulin or one directed against the C terminus of tubulin (Figures S1G-S1I). Interestingly, probing with the N-terminal-directed antibody but not the C-terminal-directed antibody also detected smaller protected protease fragments, suggesting TRiC promoted formation of partially folded N-terminal domain structures. We next incubated the prefoldin:tubulin:TRiC complex with ATP, which promotes TRiC chamber closure and prefoldin dissociation. $^{1}$ ATP incubation led to formation of full-length protease-protected tubulin, with a similar level of protection as that observed for folded $\alpha$ -and $\beta$ -tubulins in the native dimer. Since, in the presence of ATP, the TRiC lid alternates between open and closed states, we examined whether the increased protease protection was due to tubulin folding or to obstruction by the closed lid. First, we extended the time course of PK incubation to min, which should give ample time to PK to access the substrate during the open states. $^{2,2}$ Second, TRiC-tubulin was incubated with ATP to promote folding, followed by subsequent addition of apyrase for or min to hydrolyze all ATP, causing the TRiC lid to reopen. $^{1}$ Under these conditions, tubulin remained fully protease protected, confirming the ATP-induced protease protection arises from tubulin reaching the folded conformation (Figures S1G and S1I).To confirm the TRiC dependence of $\beta$ -tubulin folding, we incubated prefoldin: $\beta$ -tubulin with archaeal M. manpalaudis chaperonin, Mm-Cpn. $^{2}$ Despite its structural and mechanistic similarity To confirm the TRiC dependence of $\beta$ -tubulin folding, we incubated prefoldin: $\beta$ -tubulin with archaeal M. manpalaudis chaperonin, Mm-Cpn.2 Despite its structural and mechanistic similarity to TRiC,2 incubating the tubulin-Mm-Cpn complex with ATP/ GTP does not yield PK protection for tubulin but instead produces a digestion pattern similar to that of prefoldin: $\beta$ -tubulin alone . This experiment indicates tubulin cannot fold when encapsulated by a non-TRiC group in chaperonin. It also confirmed tubulin protease protection does not derive from a lack of PK accessibility upon substrate enclosure in a folding chamber. These experiments suggest prefoldin and TRiC differentially affect the conformation of the substrate polypeptide during ATP-dependent tubulin folding . Prefoldin maintains tubulin in a protease-sensitive unstructured state, with no obvious protease-protected fragments observed. Upon prefoldin transfer to TRiC, tubulin becomes compacted, acquiring some protease resistance, including formation of N-terminal domain structures. Finally, ATP-driven encapsulation leads to a fully protease-resistant tubulin that can bind to tubulin monomer binding factors. Since locking the TRiC chamber into a closed conformation by the addition of ATP-AlFx also led to strong protection from proteolysis, we conclude the substrate is fully encapsulated within the central chamber. We next examined folding when tubulin is directly bound to TRiC and the prefoldin-delivery step is bypassed . The LIP-PAGE assay indicated tubulin adopts the same compact intermediate upon TRiC binding, regardless of whether tubulin is copurified with TRiC or delivered via prefoldin; furthermore, ATP/GTP addition to TRiC-tubulin also yielded a fully protease-protected conformation . Thus, TRiC is primarily responsible for tubulin folding, while prefoldin functions to maintain tubulin in an unstructured state, likely to prevent formation of off-pathway trapped intermediates. We set out to study each stage of the reconstituted prefoldin:TRiC-mediated tubulin folding pathway using a hybrid structural and biochemical approach . # Defining prefoldin interactions with non-native tubulin Defining prefoldin interactions with non-native tubulinPrefoldin is a jellyfish-shaped complex with six different subunits forming a double $\beta$ sheet barrel from which coiled coils extend, forming "tentacles" . The flexible nature and relatively small size of prefoldin:tubulin hinders its structural characterization by cryo-EM. Accordingly, we mapped the prefoldin:tubulin contacts using crosslinking mass spectrometry (XL-MS), with similar results obtained for $\alpha$ -and $\beta$ -tubulins , 2B, and S2A). Substrate crosslinks (XL) mapped primarily to the coiled-coil tentacles and the terminal disordered tails of prefoldin, indicating non-native tubulin binds inside the prefoldin "chamber" (Figures 2B and S2B). Structural mapping of the  prefoldin:tubulin XL contact information suggests substrate binds prefoldin through highly heterogeneous interactions within the chamber (Figures 2C and 2D). The N-terminal domain of tubulin made numerous contacts throughout the prefoldin inner chamber, with most XL mapping to contiguous prefoldin/5/6 coiled coils. In contrast, the tubulin core domain was crosslinked primarily to one specific region in prefoldin. The C-terminal domain of tubulin also crosslinked to a continuous patch on prefoldin/5/6, while the M-domain crosslinked to several points at the bottom of the prefoldin chamber. These analyses, together with the high protease sensitivity of prefoldin-bound tubulin, suggest tubulin populates a highly dynamic conformational ensemble, even though some regions of tubulin bind more stably to specific regions within the prefoldin chamber. Prefoldin is an ATP-independent chaperone, raising the question of how substrate binding and release are regulated. Surprisingly, while binding either $\alpha$ -or $\beta$ -tubulin to prefoldin adds an additional molecular mass of $\sim$ kDa to the kDa prefoldin complex, the prefoldin-substrate complexes migrated faster on N-PAGE than prefoldin alone . We hypothesized substrate binding to prefoldin induces a conformational compaction. Native mass spectrometry analyses support this idea (Figures 2F, 2G, S2C, and S2D). Prefoldin alone produced a bimodal charge distribution for the single expected prefoldin mass (1 kDa) (Figures S2C and S2D). Because the charge states produced during gentle ionization are strongly dependent on the exposed surface area,3-3 these data indicate pure prefoldin adopts two distinct structural states: one more compact and one more extended. Analysis of prefoldin: $\beta$ -tubulin complexes by native mass spectrometry also yielded a bimodal charge series consistent with the expected mass of the binary prefoldin: $\beta$ -tubulin complex (1 kDa) (Figures 2F and S2E). Of note, the relative intensity, and therefore abundance, of the compact species was higher in prefoldin: $\beta$ -tubulin peaks compared to prefoldin peaks (Figures 2E-2G and S2E). The conclusion that tubulin binding induces a shift to the more compact prefoldin conformation supports the N-PAGE analyses as well as our finding of extensive XL between tubulin and the tentacles of PFD and PFD, located at opposite sides of the prefoldin chamber. A possible function for this compaction arises from cryo-EM structural analyses of the prefoldin-TRiC interaction, which found a flexible contact point between the base of prefoldin and the apical domain of CCT in TRiC that allows prefoldin to pivot into the chamber.1 Prefoldin adopts a compact conformation in the less engaged TRiC contact, where the two substrate binding chambers are not aligned, but shifts to an extended conformation when the prefoldin-TRiC chambers become aligned, allowing for substrate transfer (Figures 2H and 2I, S2F, and S2G). This suggests prefoldin initially binds TRiC in the compact, substrate-bound conformation, but then TRiC interactions promote the extended prefoldin state to facilitate substrate release into the chaperonin chamber. # The ternary complex of prefoldin: $\beta$ -tubulin:TRiC in the open conformation To visualize the substrate handoff from prefoldin to TRiC, we added prefoldin: $\beta$ -tubulin to purified TRiC and studied the ternary complex by cryo-EM (Figures 3A and S3A-S3D; Table S). The prefoldin: $\beta$ -tubulin:TRiC complex showed the chamber of prefoldin fully aligned with that of TRiC through subunit specific contacts . We observed a significant extra density inside the inter-ring space formed at the TRiC equatorial region not observed in the absence of substrate (Figures 3B-3D).1 The prefoldin tentacles extend into the TRiC chamber. Strikingly, the PFD coiled coil made a clear connection to the substrate-dependent density in the inter-ring space . The prefoldin in this TRiC complex is in the open extended conformation, suggesting TRiC binding reduces its affinity for the bound substrate. Indeed, no significant substrate density is observed inside the prefoldin chamber, despite the substantial extra density observed between the two TRiC rings. The oval-shaped substrate-induced extra-density is asymmetrically located within the inter-ring TRiC chamber, close to subunits CCT, 6, and. Of note, we also observe clear connections between this density and the N-terminal tails of CCT and, which are normally not resolved in open TRiC structures . To further assess the origin of the extra density, we prepared a TRiC: $\beta$ -tubulin complex generated without prefoldin and carried out cryo-EM analyses of both substrate-free apo-TRiC and TRiC: $\beta$ -tubulin. Only the $\beta$ -tubulin-containing TRiC has density in the central inter-ring region, while no notable density is A $\beta$ Tubulin relocates within chamber upon TRiC ring closure  observed in substrate-free apo-TRiC . The substrate-dependent density is located within an inter-ring chamber created by the equatorial domains of CCTs protruding into the central axis, forming a $\sim \text{nm}$ open cap covering a $\sim \times \text{nm}$ globular inner space . However, extensive focused 3D classification on the substrate-dependent extra density failed to discern any folded domain or secondary structure. While the presence of substrate-dependent density indicates the formation of a compact state, this structural uncertainty suggests heterogeneity in either the folding state or the positioning of $\beta$ -tubulin within this region of the open TriC chamber. We used XL-MS to further characterize the contacts of $\beta$ -tubulin with prefoldin and TRiC within the ternary complex and in the binary TRiC: $\beta$ -tubulin complex (Figures 3E and S2A; Table S). TRiC addition to prefoldin: $\beta$ -tubulin led to a near complete loss of $\beta$ -tubulin's XL to prefoldin and to the appearance of extensive XL to TRiC, consistent with substrate transfer between these chaperones . The ternary complex only had one $\beta$ -tubulin XL to PFD and one $\beta$ -tubulin XL to the N terminus of PFD . We also observed multiple PFD XL to the tails of CCT and CCT in the equatorial region of TRiC (Figures S3E and S3F). Together, these XL-MS observations support the structural data showing PFD inserting into the TRiC chamber placing the tip of the PFD tentacle proximal to the substrate-dependent density. Importantly, XL-MS also indicated extensive contacts between $\beta$ -tubulin and the N-and C-terminal tails extending from the equatorial domains of CCT subunits into the chamber (Figures 3F and 3G). These XL-MS findings also agree with the cryo-EM maps showing the substrate localizes to the inter-ring chamber region in close proximity to the disordered N-and C-terminal CCT tails. XL between $\beta$ -tubulin and the CCT apical domains were observed in the TRiC $\beta$ -tubulin complex generated without prefoldin but not in the prefoldin: $\beta$ -tubulin:TRiC ternary sample.1,3 We did not observe notable substrate density at the similar contour level in the apical domain regions of TRiC. Perhaps the substrate interactions with apical domains are too dynamic to be clearly observed, or alternatively, apical contacts serve in the initial capture of substrate for subsequent engagement with the CCT tails. The ternary complex with prefoldin may thus reflect the substrate state after transfer to the inter-ring chamber. While TRiC has been considered as a two-chamber folding system, our data suggest the existence of a third chamber that holds the substrate in open TRiC. Theoretically, the third interring chamber provides enough space for $\sim \text{kDa}$ protein, but it is most likely that both the substrate and the CCT tails contribute to the observed substrate-dependent density in our maps. Both TRiC and prefoldin contain long intrinsically disordered N-and C-terminal tails; the CCT tails projecting into this chamber comprise $\sim.5 \text{kDa}$ of protein mass (schematized in . The observed substrate-dependent density within TRiC, together with the increased substrate protection from PK, both indicate the substrate becomes compacted upon TRiC binding, without leading to any stable domain formation. The unstructured CCT tails contacting $\beta$ -tubulin in this region of the open inter-ring TRiC chamber contain both polar and hydrophobic character (Figures 3G and 3H). We speculate that these disordered tails function as a tethered solvent that maintains the unfolded polypeptide in a conformationally dynamic coacervate state, thus preventing the formation of trapped intermediates. This may stabilize $\beta$ -tubulin folding intermediates in a folding-competent state by interactions with CCT tails in the open chaperonin conformation (Figures 3H and 3I). # Cryo-EM identifies $\beta$ -tubulin folding intermediates within the ATP-closed TRiC chamber To examine the next step of the $\beta$ -tubulin folding cycle , we incubated the TRiC: $\beta$ -tubulin complex with ATP-AIx to stabilize the closed conformation and examined $\beta$ -tubulin folding in the chamber by cryo-EM. Following incubation with ATP-AIx, we observed open and closed TRiC states containing $\beta$ -tubulin-dependent density . As expected, the $3.8 \text{\AA}$ open TRiC state contains the $\beta$ -tubulin-dependent density in the inter-ring space (Figures, S4A, and S4B; Table S). In contrast, the closed TRiC conformation contains the substrate density in the apical region within one of the chambers , right). The repositioned $\beta$ -tubulin in the $2.8 \text{\AA}$ resolution closed TRiC density map had well-defined secondary structural element features . Additionally, most of the CCT N-and C-terminal tails could be traced in the closed conformation but not in the open conformation. In the initial overall map of closed TRiC: $\beta$ -tubulin, we visualized partially folded $\beta$ -tubulin with varying resolution, suggesting compositional and conformational heterogeneity. 3D classification focusing on the $\beta$ -tubulin density revealed four distinct conformations of $\beta$ -tubulin with structurally resolvable domains. The TRiC: $\beta$ -tubulin conformations were reconstructed to resolutions ranging from $2.9 \text{\AA}$ to $3.6 \text{\AA}$ (Figures 4B and S4C-S4E) allowing us to build models of each TRiC-bound state . In all four maps, the resolvable densities corresponded to native tubulin domains with well-resolved $\alpha$ helices and $\beta$ strands (Figures 4B and 4C; Video S). The four $\beta$ -tubulin structures are aligned along an axis of increased domain formation, suggesting $\beta$ -tubulin is progressively folded in the TRiC chamber. We also observed clear density for the negatively charged C-terminal $\beta$ -tubulin E-hook tail (Figures 4D and S5A), which is flexible in native tubulin. To assess the resolvability of our densities, we used a per-residue Q-score profile in a sliding window of  residues (Figure S4F). In agreement with their estimated resolution, we obtained consistently high Q-scores for the folded regions demonstrating the high resolvability of structural elements. In contrast, the poorly resolved segments have low Q-scores, suggestive of unfolded or conformationally dynamic regions. These conformations, designated states I-IV, reveal progressively folded $\beta$ -tubulin inside closed TRiC, with state IV corresponding to fully folded $\beta$ -tubulin (Video S). residues (Figure S4F). In agreement with their estimated resolution, we obtained consistently high Q-scores for the folded regions demonstrating the high resolvability of structural elements. In contrast, the poorly resolved segments have low Q-scores, suggestive of unfolded or conformationally dynamic regions. These conformations, designated states I-IV, reveal progressively folded $\beta$ -tubulin inside closed TRiC, with state IV corresponding to fully folded $\beta$ -tubulin (Video S).The well-resolved residues in states I-IV map to specific domains formed by discontinuous sequences suggestive of domain-wise folding of $\beta$ -tubulin (Figures 4D and 4E). The least-folded intermediate, state I, contains the folded N-terminal $\beta$ -tubulin domain (residues-1)  with the exposed surface contacting the inner wall of CCT-CCT  and the C-terminal $\beta$ -tubulin E-hook (residue-4) bound to a pocket between CCT and CCT (Figure S5A). State II contains the resolved regions of state I as well as resolvable discontinuous sequence elements corresponding to the folded C-terminal domain (C domain) of $\beta$ -tubulin, spanning residues-2 and-2 and C-terminal helices-4 (Figures 4D and 4E). The folded C-terminal domain also binds the inner wall of TRiC, contacting residues of subunits CCT and CCT . State III also contained the folded N-and C-terminal domains observed in state II as well as the discontinuous resolvable sequence elements corresponding to the folded helical core domain, spanning residues-2, 2-2, and-3 (Figures 4D and 4E). Finally, state IV resembles state III but also contains the folded middle M domain, composed of residues-3. Folding of this last M domain completes formation of a fully native $\beta$ -tubulin monomer (Figures 4D and 4E). Though the TRiC-bound $\beta$ -tubulin states exhibit C-$\alpha$ backbone variation in certain loop regions, the folded secondary structural elements in all four states are almost identical to the structure of native $\beta$ -tubulin (PDB: 6I)3 (Figure S4G). The C-terminal E-hook tail of $\beta$ -tubulin was clearly traceable in all four states nestled in a pocket formed by CCT and CCT (Figure S5A). These four structural states of $\beta$ -tubulin within the chamber of TRiC suggest a sequential folding pathway where discontinuous sequence elements fold into specific native-like domains in close association with the inner chaperonin chamber, culminating in the native state . # Domain-specific CCT contacts spatially orient and restrain $\beta$ -tubulin within the TRiC chamber Upon ATP-driven encapsulation, the inner TRiC wall engages in domain-specific contacts with all four $\beta$ -tubulin intermediates. The extent of electrostatic interactions with the inner TRiC chamber increases from the less-folded to more-folded $\beta$ -tubulin intermediates ; Video S). This suggests a different mechanism to bacterial chaperonin GroEL-ES, which promotes substrate folding by releasing encapsulated polypeptides into their central hydrophilic chamber,2 often termed an "Anfinsen cage."3 We next examined in detail the interactions between $\beta$ -tubulin and the TRiC chamber in each state. In all four $\beta$ -tubulin intermediates, negatively charged residues exposed by folded N-terminal domain contacts specific positively charged residues in CCT and CCT, while the negatively charged, normally unstructured, C-terminal E-hook tail of $\beta$ -tubulin contacts a positively charged pocket at the interface of CCT and CCT (Figures 5B and S5A; Movie S). The folded C-terminal $\beta$ -tubulin domain in states II-IV also engages in electrostatic contacts with positively charged residues in CCT and CCT. These electrostatic contacts between the folded N-and C-terminal domains and TRiC persist even once $\beta$ -tubulin reaches the fully folded state . Previous analysis recognized that the TRiC inner chamber contains an asymmetric charge distribution, with a strong positively charged inner surface contributed by subunits CCT/3/6/ 8 and a strong negatively charged inner surface contributed by CCT/4/5/7 . Of note, these charged surface patches are only formed upon lid closure following ATP hydrolysis . The $\beta$ -tubulin electrostatic interactions with the positively charged surface of TRiC segregate $\beta$ -tubulin to one side of the TRiC chamber via contacts present in all folding intermediates, while the polypeptide regions folded only in states III and IV extend into the chamber cavity (Figures 4B and S5B), suggesting additional TRiC regions contribute to $\beta$ -tubulin folding. We next examined the specific contacts between the CCT subunits and each $\beta$ -tubulin folding intermediate (Figures 5D5I). In the folded $\beta$ -tubulin N domain (blue in Figures 5D and 5E), residues on the N terminus, H, H, H, and loops H-S, H-S, and H-S form salt bridges and hydrogen bonds with residues on CCT and CCT (Figures 5D and 5E).Within the folded C domain green in Figures 5F and 5G),H and H are anchored to CCT and CCT by additional salt bridges and hydrogen bonds .Our analysis also revealed specific contacts between $\beta$ -tubulin and tails of CCT and CCT, in addition to the previously highlighted E-hook tail of $\beta$ -tubulin (Figures 5H and S5B).Remarkably, both the E-hook and the CCT tails are normally disordered, indicating that formation of the TRiC: $\beta$ -tubulin complex promotes stabilization of these  flexible regions. The negatively charged E-hook of $\beta$ -tubulin is nestled into a positively charged pocket at the interface of CCT and CCT (Figures 5H and S5A).The CCT C-terminal tail is proximal to the $\beta$ -tubulin T loop in the C domain, which is part of the GTP binding pocket i). Since the T loop forms a possible hydrogen bond to GTP, it is possible that the CCT tail facilitates formation of the tubulin GTP binding pocket. Indeed, we observe nucleotide density in state Ill of $\beta$ -tubulin . The regions corresponding to the core and M domains extend into the center of the TRiC chamber without any appreciable interactions with the inner wall. Closer examination reveals that TRiC contacts the core and M domains via the C-terminal tail of CCT in all the observed $\beta$ -tubulin folding states (Figures 5Hii and S5B).In states I and II, the C terminus of CCT interacts with unfolded core/M domain density, while in states III and IV, the CCT tail engages the N and M loops in the taxol binding site as the core and M domains form sequentially during folding (Figure S5B). This suggests the C-terminal tail of CCT interacts with and assists the folding of the core and M domains of $\beta$ -tubulin within the lumen of the chamber. Interestingly, although the flexible CCT C tail is poorly resolved in substrate-free closed TRiC chamber, we observe at a lower threshold an association between the vicinal C-terminal tails of CCT and CCT, which extend together into the chamber (Figure S5E). Consistent with this observation, the C-terminal tail of CCT is negatively charged and that of CCT is positively charged and possesses several prolines that may rigidify this flexible region. It is interesting to speculate that the C termini of CCT-CCT jointly act as a fulcrum that, upon lid closure, facilitates the repositioning of $\beta$ -tubulin to the vicinity of the CCT-CCT-CCT-CCT hemisphere. These structures indicate that all the $\beta$ -tubulin folding intermediates interact with closed TRiC through a combination of more rigid and complementary electrostatic interactions with the chamber wall and flexible interactions with the CCT tails . Importantly, the negatively charged surface character of the N-and C-terminal domains is conserved across $\alpha$ $\beta$ -and $\gamma$ -tubulins (Figure S5F), suggesting a similar mode of TRiC-facilitated folding. Emphasizing the importance of these contacts for tubulin folding, the residues within the TRiC chamber making both electrostatic inner chamber contacts and tail contacts with $\beta$ -tubulin are highly conserved (Figure S5G).It thus appears that TRiC does not release the $\beta$ -tubulin polypep tide into the closed chamber for folding. Instead, the TRiC chamber binds $\beta$ -tubulin throughout the folding process. The TRiC contacts directly restrain and orient the N domain and C-terminal tail of $\beta$ -tubulin to one hemisphere in the chamber, thus limiting the folding polypeptide's degree of freedom and preventing its free rotation in the chamber. TRiC interactions shaping the tubulin folding landscape Our four cryo-EM structures of progressively folded $\beta$ -tubulin (Figure) demonstrate the involvement of different CCT subunits at different folding intermediates via specific chemical interactions and the unexpected roles of the intrinsically disordered CCT tails guiding the unstructured domains of $\beta$ -tubulin toward the fully folded structure . A pathway invoking early folding of the N-terminal $\beta$ -tubulin domain, which is well resolved in all four TRiC-bound folding intermediates, resonates with the PK-sensitivity assays of the folding reaction, which indicate early protection of an N-terminal domain but rapid digestion of the C terminus until completion of full-length $\beta$ -tubulin folding (Figures S1G-S1l). Our results suggest how TRiC-directed $\beta$ -tubulin folding circumvents the challenges posed by its complex topology and aggregation-prone $\beta$ sheets. The progressively folded intermediates observed in the chamber form domains that are discontinuous in sequence but orchestrate the continuous formation of a hydrophobic $\beta$ sheet running the length of the $\beta$ -tubulin structure ; Movie S).Anchoring the surface of the folded N and C domains to the TRiC chamber wall not only establishes the native topology but also positions the growing folding intermediates so that their hydrophobic core extends toward the interior of the chamber ii). Comparing states I through IV, each increasingly folded intermediate adds another $\beta$ strand layer to the exposed hydrophobic core, likely priming it for the next folding step. Importantly, each intermediate also incorporates the helices above and below the growing $\beta$ sheet, thus protecting its edges and reducing the potential for off-pathway interactions. In state I, the folded N domain exposes the hydrophobic strands S and S. Completion of the Rossman fold in state II allows helices H, H, and H to cover the newly incorporated strands S and S.3 Formation of the core domain in state Ill reduces a change in direction between strands S and S, mediated by helix H. The CCT tail may participate in this step, as it interacts with this region of the core domain. The last folding step to reach native $\beta$ -tubulin involved completion of the $\beta$ sheet by  addition of sheets S-S and positioning of helices H, H, and H to cover the hydrophobic core of tubulin. We propose that these TRiC interactions shape the tubulin folding landscape to establish the native topology and preclude the formation of trapped folding intermediates, thereby reducing the frustration of the folding landscape. # TRiC action in folding pathway in the context of tubulin evolution While eukaryotic tubulin folding exhibits an obligate requirement for TRiC, its prokaryotic homolog FtsZ can spontaneously fold without chaperones.4,4 Both proteins share a similar overall topology with other superfamily members in prokaryotes and archaea, including the overall domain organization and the extended core $\beta$ sheet (Figures 6C, S6A, and S6B).4,4-4 However, eukaryotic tubulins are distinguished by additional features integral to their unique function and regulation, including remodeling of GTP affinity and hydrolysis kinetics, the ability to engage in microtubule lateral contacts, and binding microtubule-associated proteins (MAPs).4,4-4 These tubulin functions are accompanied by insertions in the H-S loop and two negatively charged helices, H and H; by the presence of the E-hook tail; by the remodeling of the residues contacting GTP; and by extensive changes to the C domain leading to an increase in hydrophobic interdomain interfaces,4-4 (highlighted in  for 612l vs 1FSZ,4 S6C). Eukaryotic tubulins are also distinguished by the strongly negative surface charge on the folded N and C domains as well as a negatively charged C-terminal E-hook tail. We hypothesize these features of eukaryotic tubulins hinder their ability to fold spontaneously or with simpler chaperone systems. Remarkably, the tubulin-specific insertions correspond to the sites of TRiC binding during folding in the closed chamber, suggesting their incorporation into ancestral tubulin was accompanied by a need for specialized folding assistance . This hypothesis is supported by close examination of how TRiC mediates formation of the unique N-C domain interface in $\beta$ -tubulin (Figures 6E and S6D). The T loop, which participates in the folded $\beta$ -tubulin N-C interdomain interface, shifts between two alternative conformations in the TRiC-bound $\beta$ -tubulin folding intermediates, with $1\dot{\mathsf{A}}\mathsf{C} -\alpha$ variation (Figures S4G and S5C; Video S). Our maps' good Q-score resolve the bulky side chains in the T loop, such as H and Y, allowing us to trace the density of T in the folding intermediates (Figure S6D). The H-S loop forms charged and hydrogen-bonded interactions with the TRiC chamber, suggesting a stabilizing effect on the inserted $\beta$ -tubulin loops by the TRiC chamber (Figures 4E and 6D). In state I, the T loop, resolved at residues-1, adopts an extended conformation directed by contacts with E of CCT and D of CCT (Figure S5C). In state II, a decrease in resolvability of residues-1 in T (Figure S5C) suggests the loop changes conformation upon folding of the C domain. In states III and IV, the T loop becomes well resolved due to its stabilization by interaction with the folded C domain, adopting the native $\beta$ -tubulin conformation (PDB: 6I)6 (Figures S4G and S5C). Accordingly, N of T serves in GTP $\gamma$ -phosphate sensing, while hydrophobic T residues including W are embedded in the hydrophobic interface between the N and C domains . Folding of the core helix domain completes the GTP binding pocket by contributing the H-H, consistent with the nucleotide density in the GTP binding pocket observed in states III and IV (Figures 5D, 6E, and S5C). Thus, TRiC contacts keep the T loop in an extended conformation when the C domain is not folded, and these contacts are released to allow the native conformation at the interdomain interface upon C domain folding. # DISCUSSION Why some proteins have an obligate requirement for TRiC assistance has long been enigmatic. We combined XL-MS and cryo-EM to elucidate the pathway delivering unstructured $\beta$ -tubulin from PFD to the TRiC chamber where it folds through specific contacts between the substrate and the CCT subunits . We determined near-atomic-resolution structures of multiple stable folding intermediates of $\beta$ -tubulin engaged in specific interactions with various CCT subunits including the intrinsically disordered termini. Our data suggest that these interactions guide tubulin to the native state through stepwise formation of partially folded intermediates . Throughout this pathway, the substrate is not released into the central TRiC chamber, unlike what is observed with GroEL-ES, but instead remains anchored through specific contacts with the CCT wall and the C tails. These findings establish that, for tubulin, and perhaps other TRiC substrates, the folding pathway is guided by explicit interactions with the inner TRiC chamber. Of note, a recent study published during the review process. (NSMB, doi: 1.1/s-0-0-1) reports that TRiC immunoprecipitated from a cell lysate contains a partially folded tubulin molecule within the closed chamber in the same location we observed for our averaged initial TRiC-ATP-tubulin folding intermediate in . It is tempting to speculate that the tubulin contained within TRiC isolated from cells became associated during the process of biogenesis and thus represents the ensemble states of biologically meaningful folding intermediates. While further experiments should probe how tubulin folds in vivo following translation, these data suggest our reconstituted folding reaction recapitulates the intermediates populated during biological folding of tubulin. # Beyond Anfinsen: The TRiC chamber shapes folding intermediates Beyond Anfinsen: The TRiC chamber shapes folding intermediatesPrefoldin-bound $\beta$ -tubulin is delivered to the open TRiC chamber, where interactions with the disordered CCT tails generates a compact entity without discernible folded elements. We speculate the disordered tails confined within the TRiC chamber act as a tethered solvent to maintain the fluidity of the substrate promoting a dynamic yet compacted state. The liquid-like nature of such a coacervate may prevent the formation of non-productive or trapped intermediates, priming the substrate for productive folding upon chamber closure . Upon ATP-dependent chamber closure, the substrate repositions from the equatorial chamber into one of the ring-enclosed folding chambers . The ATP-closed TRiC chamber directs the stepwise formation of $\beta$ -tubulin folding intermediates that remain bound to specific regions in the chamber through electrostatic and H-bond interactions between $\beta$ -tubulin and the TRiC chamber. These contacts establish the necessary native topology for $\beta$ -tubulin and help orchestrate a hydrophobic collapse to then drive sequential folding of the $\beta$ -tubulin domains. One feature of the discontinuous formation of $\beta$ -tubulin domains in this folding pathway is the unidirectional assembly of the $\beta$ sheet spanning the entire hydrophobic core in folded $\beta$ -tubulin. These TRiC contacts appear to direct $\beta$ -tubulin folding through sequential domain formation without releasing the substrate into the chamber. This contrasts to the mechanism of chaperonin GroEL-ES, which releases polypeptides into an inner chamber that provides a unique confined environment for folding.4-5 The fact that contacts with the closed TRiC chamber guide multiple steps in $\beta$ -tubulin folding raises doubts on the universality of Anfinsen's principle. The concept of TRiC directing the folding pathway is illustrated by the notable structural change in the T loop of $\beta$ -tubulin between states I and II. TRiC keeps the T-loop in an extended conformation through contacts with the inner wall until the C-terminal domain folds through association with the TRiC wall, then the T loop is released from the chamber wall to become nestled at the N-C domain interface. The T loop acts as a folding switch to control sequential folding of the GTP binding pocket after completion of the N-C domain interface . Thus, TRiC binding to the T loop ensures the proper folding of the C domain, establishing the overall fold topology of $\beta$ -tubulin and allowing the hydrophobic core of the M and core domains to fold. Perhaps GTP binding to the incipient nucleotide pocket drives these subsequent steps, as observed for bacterial tubulin FtsZ , where GTP binding is a limiting step during spontaneous refolding.5 The TRiC: $\beta$ -tubulin contacts directly orchestrate the progression of events needed to form the GTP binding pocket by sequential positioning of the nucleotide sensing T loop to form the N-C domain interface, which are not conserved in FtsZ (Figures 6C-6E). Consistent with this, tubulin mutations in the N-C domain interface affect $\beta$ -tubulin folding and release from TRiC (Figure S6E). While TRiC appears to direct the folding landscape through formation of specific intermediates , the tubulin-CCT contacts may also disfavor formation of kinetically trapped states . # Insights into evolution of eukaryotic proteins that cannot fold without TRiC It is intriguing to speculate why and how this complex TRiCtubulin requirement evolved from simpler ancestral proteins like FtsZ, which can refold without assistance. Compared with FtsZ, tubulin contains a series of structural changes that serve to enable functional interactions at the lateral interface of protofilaments and binding of microtubule-associated proteins (MAPs) and kinosin,4,5 while these additional elements support the novel properties of eukaryotic tubulin, they appear to impair spontaneous folding,5 perhaps leading to off-pathway or kinetically trapped intermediates . Strikingly, the major sites of $\beta$ -tubulin interaction with the TRiC-chamber are in these eukaryote-specific regions.We propose these eukaryotic-specific changes in tubulin evolved to exploit the hetero-oligomeric and asymmetric nature of TRiC to circumvent the inherent challenges they pose to the folding pathway. The finding that tubulin folding is closely directed by TRiC chamber interactions opens new perspectives to understand the role of TRiC in eukaryotic protein evolution. The striking expansion of TRiC subunits directs the subunit-specific diversification of substrate binding sites, ATP affinity, and chamber properties, all of which contribute to its unique ability to fold some eukaryotic proteins. In turn, these may underlie their obligate requirement for TRiC to fold. Given that TRiC facilitates folding of $\sim\%$ of the proteome, it is likely that some proteins employ TRiC like an Anfinsen cage, similar to GroEL-ES, to prevent aggregation and smooth the folding landscape.5 Indeed, the now more limited space present in the chamber upon ATP-induced lid closure may restrain the possible orientations or conformations of the encapsulated polypeptide, significantly reducing the structural ensembles available to the substrate within the chamber of closed TRiC. Other proteins may additionally exploit other evolved features of the TRiC hetero-oligomer for which the diverse apical domain sequence and structure can provide substrate interaction specificity to promote a specific polypeptide binding topology.1,1,1,2 Also, some substrates may interact distinctly with specific regions in the chamber.5 Our finding that the eukaryote-specific tubulin insertions evolved surface-exposed motifs that bind specific regions in the TRiC chamber to shape the folding pathway provides an intriguing clue as to how some eukaryotic proteins evolved to exploit the unique properties of TRiC to acquire new functionalities that would otherwise impede their correct folding. # Limitations of the study Limitations of the studyA TRiC-directed $\beta$ -tubulin folding pathway that progresses from less folded to more folded states is here derived from snapshot images at single-equilibrium condition. Time-resolved kinetic analyses of tubulin folding, such as time-resolved H-D exchange and spectroscopic approaches are needed to validate and extend this pathway. The observation that the CCT intrinsically disordered tails promote TRiC-mediated folding resonates with findings for other chaperones that also use disordered domains for function.5,6 Extending the idea that regions of intrinsic disorder promote folding will open new perspectives on what constitutes a chaperone and revise the classic notion that chaperones only function through domains that recognize non-native proteins through hydrophobic surfaces. # STARMETHODS Detailed methods are provided in the online version of this paper and include the following: KEY RESOURCES TABLE RESOURCE AVAILABILITY $\bigcirc$ Lead contact $\bigcirc$ Materials availability $\bigcirc$ Data and code availability EXPERIMENTAL MODEL AND SUBJECT DETAILS METHOD DETAILS $\bigcirc$ Cloning $\bigcirc$ Protein expression and purification $\bigcirc$ Native and SDS PAGE analysis $\bigcirc$ Tubulin folding assays $\bigcirc$ Proteinase K assays $\bigcirc$ Crosslinking-mass spectrometry $\bigcirc$ Native Mass spectrometry $\bigcirc$ Cryo-EM specimen preparation and data collection $\bigcirc$ Data processing and 3D refinement $\bigcirc$ Model building $\bigcirc$ PISA analysis $\bigcirc$ Logo plot generation $\bigcirc$ Sequence alignment and phylogenetic tree building QUANTIFICATION AND STATISTICAL ANALYSIS # SUPPLEMENTAL INFORMATION Supplemental information can be found online at https://doi.org/1.1/j.cell.2.1.0. # ACKNOWLEDGMENTS Supported by NIH grants R01GM to J.F. and P41GM, S10OD, and R01GM to W.C. and the Korean National Research Foundation (2019R1C1C, 2020R1A5A, 2021M3A, 2019M3E5D) and SUHF foundation to S.-H.R. Cryo-EM data collected at the Stanford-SLAC CryoEM facilities. We thank Dr. P. Picotti and the UCSF Mass Spectrometry Facility for access to instrumentation. # AUTHOR CONTRIBUTIONS U.F., D.G., W.C., and S.-H.R. conceived the project; D.G. carried out all cloning and biochemical experiments; B.M. and Y.Z. collected cryo-EM datasets. Y.Z. and J.P. carried out cryo-EM data analysis and focused classification. D.G. and A.L. carried out XL-MS experiments; M.C. carried out native mass spectrometry; G.P. generated Video S, D.G., Y.Z., J.F., J.P., W.C., and S.-H.R. wrote the manuscript. All authors contributed to the final manuscript. # DECLARATION OF INTERESTS The authors declare no competing interests. Received: February, 2 Revised: September, 2 Accepted: November, 2 Published: December, 2 Corrected online: April, 2 # REFERENCES. Buchner, J. (2). Molecular chaperones and protein quality control: an introduction to the JBC Reviews thematic series. J. Biol. Chem. 2, 2-2. https://doi.org/1.1/jbc.rev.1.0. 2. Hartl, F.U., Bracher, A., and Hayer-Hartl, M. (2). Molecular chaperones in protein folding and proteostasis. Nature, 3-3. https://doi.org/1.1/nature. 3. Labbadia, J., and Morimoto, R.I. (2). The biology of proteostasis in aging and disease. Annu. Rev. Biochem. 9, 1-1. https://doi.org/1.1/annurev-biochem-0-0. 4. Mogk, A., Bukau, B., and Kampinga, H.H. (2). Cellular Handling of Protein Aggregates by Disaggregation Machines. Mol Cell, 2-2. https://doi.org/1.1/j.molcel.2.0.0. 5. Plate, L., Paxman, R.J., Wiseman, R.L., and Kelly, J.W. (2). Modulating protein quality control. Elife, e. https://doi.org/1.7/elife.1. 6. Deuerling, E., and Bukau, B. (2). Chaperone-assisted folding of newly synthesized proteins in the cytosol. Crit. Rev. Biochem. Mol. Biol. 3, 2-2. https://doi.org/1.1/1. 7. Deuerling, E., Gamerdinger, M., and Kreft, S.G. (2). Chaperone Interactions at the Ribosome. Cold Spring Harb. Perspect Biol, a. https://doi.org/1.1/cshperspect.a. 8. Willmund, F., del Alamo, M., Pechmann, S., Chen, T., Albanese, V., Dammer, E., Peng, J., and Frydman, J. (2). The cotranslational function of ribosome-associated Hsp in eukaryotic protein homeostasis. Cell, 1-2. https://doi.org/1.1/j.cell.2.1.0. 9. Yam, A.Y., Xia, Y., Lin, H.T.J., Burlingame, A., Gerstein, M., and Frydman, J. (2). Defining the TRiC/CCT interactome links chaperonin function to stabilization of newly made proteins with complex topologies. Nat. Struct. Mol. Biol. 1, 1-1. https://doi.org/1.1/nsmb.1. 1. Stein, K.C., Kriel, A., and Frydman, J. (2). Nascent Polypeptide Domain Topology and Elongation Rate Direct the Cotranslational Hierarchy of Hsp and TRiC/CCT. Mol Cell, 1-1.e. https://doi.org/1.1/j.molcel.2.0.0. 1. Gestaut, D., Roh, S.H., Ma, B., Pintilie, G., Joachimiak, L.A., Leitner, A., Walzthoeni, T., Aebersold, R., Chiu, W., and Frydman, J. (2). The Chaperonin TRiC/CCT Associates with Prefoldin through a Conserved Electrostatic Interface Essential for Cellular Proteostasis. Cell, 7-7.e. https://doi.org/1.1/j.cell.2.0.0. 1. Balchin, D., Milicic, G., Strauss, M., Hayer-Hartl, M., and Hartl, F.U. (2). Pathway of Actin Folding Directed by the Eukaryotic Chaperonin TRiC. Cell, 1-1.e. https://doi.org/1.1/j.cell.2.0.0. 1. Balchin, D., Hayer-Hartl, M., and Hartl, F.U. (2). In vivo aspects of protein folding and quality control. Science, aac. https://doi.org/1.1/science.aac. 1. Cong, Y., Baker, M.L., Jakana, J., Woolford, D., Miller, E.J., Reissmann, S., Kumar, R.N., Redding-Johanson, A.M., Batth, T.S., Mukhopadhyay, A., Ludtke, S.J., Frydman, J., and Chiu, W. (2). 4.0-A resolution cryo-EM structure of the mammalian chaperonin TRiC/CCT reveals its unique subunit arrangement. Proc. Natl. Acad. Sci. USA. 1, 4-4. https://doi.org/1.1/pnas.0. 1. Leitner, A., Joachimiak, L., Bracher, A., Monkemeyer, L., Walzthoeni, T., Chen, B., Pechmann, S., Holmes, S., Cong, Y., Ma, B., Ludtke, S., Chiu, W., Hartl, F., Aebersold, R., and Frydman, J. (2). The molecular architecture of the eukaryotic chaperonin TRiC/CCT. Structure, 8-8. 1. Joachimiak, L., Walzthoeni, T., Liu, C.W., Aebersold, R., and Frydman, J. (2). The structural basis of substrate recognition by the eukaryotic chaperonin TRiC/CCT. Cell, 1-1. https://doi.org/1.1/j.cell.2.1.0. 1. Cong, Y., Schroder, G.F., Meyer, A.S., Jakana, J., Ma, B., Dougherty, M.T., Schmid, M.F., Reissmann, S., Levitt, M., Ludtke, S.L., Frydman, J., and Chiu, W. (2). Symmetry-free cryo-EM structures of the chaperonin TRiC along its ATPase-driven conformational cycle. EMBO J. 3, 7-7. https://doi.org/1.1/emboj.2.3. 1. Reissmann, S., Joachimiak, L., Chen, B., Meyer, A., Nguyen, A., and Frydman, J. (2). A gradient of ATP affinities generates an asymmetric power stroke driving the chaperonin TRiC/CCT folding cycle. Cell Rep. 2, 8-8. https://doi.org/1.1/j.celrep.2.0.0. 1. Roh, S.H., Kasembeli, M., Galaz-Montoya, J.G., Trnka, M., Lau, W.C.Y., Burlingame, A., Chiu, W., and Tweardy, D.J. (2). Chaperonin TRiC/ CCT Modulates the Folding and Activity of Leukemogenic Fusion Oncoprotein AML-ETO. J. Biol. Chem. 2, 4-4. https://doi.org/1.1/jbc.m.6. 2. Spiess, C., Miller, E.J., McClellan, A.J., and Frydman, J. (2). Identification of the TRiC/CCT substrate binding sites uncovers the function of subunit diversity in eukaryotic chaperonins. Mol Cell, 2-3. https://doi.org/1.1/j.molcel.2.0.0. 2. Tam, S., Spiess, C., Auyeung, W., Joachimiak, L., Chen, B., Poirier, M.A., and Frydman, J. (2). The chaperonin TRiC blocks a huntingin sequence element that promotes the conformational switch to aggregation. Nat. Struct. Mol. Biol. 1, 1-1. https://doi.org/1.1/nsmb.1. 2. Geissler, S., Siegers, K., and Schiebel, E. (1). A novel protein complex promoting formation of functional alpha-and gamma-tubulin. EMBO J. 1, 9-9. https://doi.org/1.1/emboj/1.4.9. 2. Vainberg, I.E., Lewis, S.A., Rommelaere, H., Ampe, C., Vandekerckhove, J., Klein, H.L., and Cowan, N.J. (1). Prefoldin, a chaperone that delivers unfolded proteins to cytosolic chaperonin. Cell, 8-8. https://doi.org/1.1/s-8(0)8-4. 2. Lewis, S.A., Tian, G., and Cowan, N.J. (1). The alpha-and beta-tubulin folding pathways. Trends Cell Biol. 7, 4-4. https://doi.org/1.1/s-8(9)0-9. 2. Nithianantham, S., Le, S., Seto, E., Jia, W., Leary, J., Corbett, K.D., Moore, J.K., and Al-Bassam, J. (2). Tubulin cofactors and Arl are cage-like chaperones that regulate the soluble $\alpha \beta$ -tubulin pool for microtubule dynamics. Elife, e. https://doi.org/1.7/elife.0. 2. Frydman, J., Nimmesgem, E., Ohtsuka, K., and Hartl, F.U. (1). Folding of nascent polypeptide chains in a high molecular mass assembly with molecular chaperones. Nature, 1-1. https://doi.org/1.1/370111a. 2. Thulasiraman, V., Ferreyra, R.G., and Frydman, J. (2). Folding assays. Assessing the native conformation of proteins. Methods Mol. Biol. 1, 1-1. https://doi.org/1.1/1-5-0-6:1. 2. Meyer, A.S., Gillespie, J.R., Walther, D., Millet, I.S., Doniach, S., and Frydman, J. (2). Closing the folding chamber of the eukaryotic chaperonin requires the transition state of ATP hydrolysis. Cell, 3-3. https://doi.org/1.1/s-8(0)0-6. 2. Reissmann, S., Parnot, C., Booth, C.R., Chiu, W., and Frydman, J. (2). Essential function of the built-in lid in the allosteric regulation of eukaryotic and archaeal chaperonins. Nat. Struct. Mol. Biol. 1, 4-4. https://doi.org/1.1/nsmb. 3. Collier, M.P., Alderson, T.R., de Villiers, C.P., Nicholls, D., Gastall, H.Y., Allison, T.M., Degiacomi, M.T., Jiang, H., Mlynek, G., Furst, D.O., et al. (2). HspB phosphorylation regulates its intramolecular dynamics and mechanosensitive molecular chaperone interaction with filamin C. Sci. Adv. 5, eaav. https://doi.org/1.1/sciadv.aav. 3. Kaltashov, I.A., and Molimen, A. (2). Estimates of protein surface areas in solution by electrospray ionization mass spectrometry. Anal. Chem. 7, 5-5. https://doi.org/1.1/ac+. 3. Santambrogio, C., Frana, A.M., Natalello, A., Papaleo, E., Regonesi, M.E., Doglia, S.M., Tortora, P., Invernizzi, G., and Grandori, R. (2). The role of the central flexible region on the aggregation and conformational properties of human ataxin-3. FEBS J. 2, 4-4. https://doi.org/1.1/j.1-4.2.0. x. 3. Scott, D., Layfield, R., and Oldham, N.J. (2). Ion mobility-mass spectrometry reveals conformational flexibility in the deubiquitinating enzyme USP. Proteomics, 2-2. https://doi.org/1.1/pmic.2. 3. Cuellar, J., Ludlam, W.G., Tensmeyer, N.C., Joba, T., Dhavale, M., Santiago, C., Bueno-Carrasco, M.T., Mann, M.J., Pimpton, R.L., Makaju, A., Franklin, S., Willardson, B.M., and Valpuesta, J.M. (2). Structural and functional analysis of the role of the chaperonin CCT in mTOR complex assembly. Nat. Commun. 1, 2. https://doi.org/1.1/s-0-1-1. 3. Pintilie, G., Zhang, K., Su, Z., Li, S., Schmid, M.F., and Chiu, W. (2). Measurement of atom resolvability in cryo-EM maps with Q-scores. Nat. Methods, 3-3. https://doi.org/1.1/s-0-0-1. 3. Adib, R., Montgomery, J.M., Atherton, J., O'Regan, L., Richards, M.W., Straatman, K.R., Roth, D., Straube, A., Bayliss, R., Moores, C.A., and Fry, A.M. (2). Mitotic phosphorylation by NEK and NEK reduces the microtubule affinity of EML to promote chromosome congression. Sci. Signal. 1, eaaw. https://doi.org/1.1/scisignal.aaw. 3. Ellis, R.J. (1). Revisiting the Anfinsen cage. Fold Des, R-R. https://doi.org/1.1/s-0(9)0-1. 3. Orr, G.A., Verdier-Pinard, P., McDaid, H., and Horwitz, S.B. (2). Mechanisms of Taxol resistance related to microtubules. Oncogene, 7-7. https://doi.org/1.1/sj.onc.1. 3. Rossmann, M.G., Moras, D., and Olsen, K.W. (1). Chemical and biological evolution of a nucleotide-binding protein. Nature, 1-1. https://doi.org/1.1/250194a. 4. Andreu, J.M., Oliva, M.A., and Monasterio, O. (2). Reversible Unfolding of FtsZ Cell Division Proteins from Archaea and Bacteria. J. Biol. Chem. 2, 4-4. https://doi.org/1.1/jbc.m. 4. Erickson, H.P., Anderson, D.E., and Ossawa, M. (2). FtsZ in bacterial cytokinesis: cytoskeleton and force generator all in one. Microbiol. Mol. Biol. Rev. 7, 5-5. https://doi.org/1.1/mmbr.0-1. 4. Aylett, C.H.S., and Duggin, I.G. (2). The Tubulin Superfamily in Archaea. Subcell. Biochem. 8, 3-4. https://doi.org/1.1/9-3-3-5-5_. 4. Nogales, E., Downing, K.H., Amos, L.A., and Lowe, J. (1). Tubulin and FtsZ form a distinct family of GTPases. Nat. Struct. Biol. 5, 4-4. https://doi.org/1.1/nsb-4. 4. Yutin, N., and Koonin, E.V. (2). Archaeal origin of tubulin. Biol. Direct, 1. https://doi.org/1.1/1-6-7-1. 4. Janke, C., and Chlo Bulinski, J. (2). Post-translational regulation of the microtubule cytoskeleton: mechanisms and functions. Nat. Rev. Mol. Cell Biol. 1, 7-7. https://doi.org/1.1/nrm. 4. Nogales, E., Wolf, S.G., and Downing, K.H. (1). Structure of the $\alpha \beta$ tubulin dimer by electron crystallography. Nature, 1-2. https://doi.org/1.1/3. 4. Skiniotis, G., Cochran, J.C., Muller, J., Mandelkow, E., Gilbert, S.P., and Haengar, A. (2). Modulation of krinsin binding by the C termini of tubulin. EMBO J. 2, 9-9. https://doi.org/1.1/sj.emboj.7. 4. Lowe, J., and Amos, L.A. (1). Crystal structure of the bacterial cell-division protein FtsZ. Nature, 2-2. https://doi.org/1.1/3. 4. Chen, D.H., Madan, D., Weaver, J., Lin, Z., Schrder, G., Chiu, W., and Rye, H. (2). Visualizing GroEL/ES in the act of encapsulating a folding protein. Cell, 1-1. https://doi.org/1.1/j.cell.2.0.0. 5. Fenton, W.A., and Horwich, A.L. (1). GroEL-mediated protein folding. Protein Sci. 6, 7-7. https://doi.org/1.1/pro.5. 5. Gupta, A.J., Haldar, S., Milicic, G., Hartl, F.U., and Hayer-Hartl, M. (2). Active cage mechanism of chaperonin-assisted protein folding demonstrated at single-molecule level. J. Mol. Biol. 4, 2-2. https://doi.org/1.1/j.jmb.2.0.0. 5. Motojima, F. (2). How do chaperonins fold protein? Biophysics, 9-1. https://doi.org/1.2/biophysics.1.9. 5. Sharma, S., Chakraborty, K., Muller, B.K., Astola, N., Tang, Y.C., Lamb, D.C., Hayer-Hartl, M., and Hartl, F.U. (2). Monitoring protein conformation along the pathway of chaperonin-assisted folding. Cell, 1-1. https://doi.org/1.1/j.cell.2.0.0. 5. Huecas, S., Canosa-Valls, A.J., Araujo-Bazan, L., Ruiz, F.M., Laurents, D.V., Fernandez-Tornero, C., and Andreu, J.M. (2). Nucleotide-induced folding of cell division protein FtsZ from Staphylococcus aureus. FEBS J. 2, 4-4. https://doi.org/1.1/febs.1. 5. Wang, Y., Tian, G., Cowan, N.J., and Cabral, F. (2). Mutations affecting beta-Asulin folding and degradation. J. Biol. Chem. 2, 1-1. https://doi.org/1.1/jbc.m. 5. Bertrand, S., Barthelemy, L., Oliva, M.A., Carrascosa, J.L., Andreu, J.M., and Valpuesta, J.M. (2). Folding, stability and polymerization properties of FtsZ chimeras with inserted tubulin loops involved in the interaction with the cytosolic chaperonin CCT and in microtubule formation. J. Mol. Biol. 3, 3-3. https://doi.org/1.1/j.jmb.2.1.0. 5. Georgescauld, F., Popova, K., Gupta, A., Bracher, A., Engen, J., Hayer-Hartl, M., and Hartl, F. (2). GroEL/ES chaperonin modulates the mechanism and accelerates the rate of TIM-barrel domain folding. Cell, 9-9. https://doi.org/1.1/j.cell.2.0.0. 5. Knowlton, J.J., Gestaut, D., Ma, B., Taylor, G., Seven, A.B., Leitner, A., Wilson, G.J., Shanker, S., Yates, N.A., Prasad, B.V.V., Aebersold, R., Chiu, W., Frydman, J., and Dermody, T.S. (2). Structural and functional dissection of reovirus capsid folding and assembly by the prefoldin-TRiC/ CCT chaperone network. Proc. Natl. Acad. Sci. USA. 1, e. https://doi.org/1.1/pnas.2. 5. Quan, S., Wang, L., Petrotchenko, E.V., Makepeace, K.A., Horowitz, S., Yang, J., Zhang, Y., Borchers, C.H., and Bardwell, J.C. (2). Super Spy variants implicate flexibility in chaperone action. Elife, e. https://doi.org/1.7/eLife.0. 6. Reichmann, D., Xu, Y., Cremers, C., Ilbert, M., Mittelman, R., Fitzgerald, M., and Jakob, U. (2). Order out of disorder: working cycle of an intrinsically unfolded chaperone. Cell, 9-9. https://doi.org/1.1/j.cell.2.0.0. 6. Leitner, A., Walzthoeni, T., and Aebersold, R. (2). Lysine-specific chemical cross-linking of protein complexes and identification of cross-linking sites using LC-MS/MS and the xQuest/xProphet software pipeline. Nat. Protoc. 9, 1-1. https://doi.org/1.1/nprot.2.1. 6. Graham, M., Combe, C., Kolbowski, L., and Rappsilber, J. (2). xView: A common platform for the downstream analysis of Crosslinking Mass Spectrometry data. Preprint at bioRxiv. https://doi.org/1.1/5. 6. Zivanov, J., Nakane, T., Forsberg, B.O., Kimanius, D., Hagen, W.J., Lindahl, E., and Scheres, S.H. (2). New tools for automated high-resolution cryo-EM structure determination in RELION-3. Elife, e. https://doi.org/1.7/elife.4. 6. Punjani, A., Rubinstein, J.L., Fleet, D.J., and Brubaker, M.A. (2). cryo-SPARC: algorithms for rapid unsupervised cryo-EM structure determination. Nat. Methods, 2-2. https://doi.org/1.1/nmeth.4. 6. Zheng, S.Q., Palovcak, E., Armache, J.P., Velisa, K.A., Cheng, Y., and Agard, D.A. (2). MotionCor: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. Nat. Methods, 3-3. https://doi.org/1.1/nmeth.4. 6. Zhang, R. (2). Gcti: Real-time CT determination and correction. J. Struct. Biol. 1, 1-1. https://doi.org/1.1/j.jsb.2.1.0. 6. Emsley, P., Lohkamp, B., Scott, W.G., and Cowtan, K. (2). Features and development of Coot. Acta Crystallogr D Biol Crystallogr, 4-5. https://doi.org/1.1/s. 6. Kidmose, R.T., Juhl, J., Nissen, P., Boesen, T., Karlsen, J.L., and Pedersen, B.P. (2). Namdinator -automatic molecular dynamics flexible fitting of structural models into cryo-EM and crystallography experimental maps. IUCrJ, 5-5. https://doi.org/1.1/s. 6. Afonine, P.V., Klaholz, B.P., Moriarty, N.W., Poon, B.K., Sobolev, O.V., Terwilliger, T.C., Adams, P.D., and Urzhumtsev, A. (2). New tools for the analysis and validation of cryo-EM maps and atomic models. Acta Crystallogr D Struct Biol, 8-8. https://doi.org/1.1/s. 7. Pettersen, E.F., Goddard, T.D., Huang, C.C., Meng, E.C., Couch, G.S., Croll, T.I., Morris, J.H., and Ferrin, T.E. (2). LCsF ChimeraX: Structure visualization for researchers, educators, and developers. Protein Sci. 3, 7-8. https://doi.org/1.1/pro.3. 7. Dey, K.K., Xie, D., and Stephens, M. (2). A new sequence logo plot to highlight enrichment and depletion. BMC Bioinf. 1, 4. https://doi.org/1.1/s-0-2-3. 7. Di Tommaso, P., Moretti, S., Xenarios, I., Orobitg, M., Montanyola, A., Chang, J.M., Taly, J.F., and Notredame, C. (2). T-Coffee: a web server for the multiple sequence alignment of protein and RNA sequences using structural information and homology extension. Nucleic Acids Res. 3, W-W. https://doi.org/1.1/nar/gkr. 7. Marty, M.T., Baldwin, A.J., Marklund, E.G., Hochberg, G.K.A., Benesch, J.L.P., and Robinson, C.V. (2). Bayesian Deconvolution of Mass and Ion Mobility Spectra: From Binary Interactions to Polydisperse Ensembles. Anal. Chem. 8, 4-4. https://doi.org/1.1/acs.analchem.5b. 7. Hall, B.G. (2). Building phylogenetic trees from molecular data with MEGA. Mol Biol E, 1-1. https://doi.org/1.1/molbev/mst. # STARMETHODS # KEY RESOURCES TABLE <table><tr><td>REAGENT or RESOURCE</td><td>SOURCE</td><td>IDENTIFIER</td></tr><tr><td>Antibodies</td><td></td><td></td></tr><tr><td>anti -tubulin (Mouse)</td><td>ProteinTech</td><td>Cat# 6-1-kg, RRID: AB</td></tr><tr><td>anti -tubulin (Rabbit) Polyclonal</td><td>Abcam</td><td>Cat# AB, RRID: AB_</td></tr><tr><td>anti -tubulin (Mouse) Monoclonal C-term AA epitope</td><td>Millipore Sigma</td><td>Cat# 0-6, RRID: AB_</td></tr><tr><td>anti -tubulin (Mouse) Monoclonal</td><td>Sigma-Aldrich</td><td>Cat# T, RRID: AB_</td></tr><tr><td>Custom anti human PFD (Rabbit)</td><td>Cocalico Biologicals</td><td>N/A</td></tr><tr><td>IRDye 680RD Goat anti-mouse IgG</td><td>LiCOr</td><td>Cat# 9-6, RRID: AB_</td></tr><tr><td>IRDye 800RD Donkey anti-mouse IgG</td><td>LiCOr</td><td>Cat# 9-3, RRID: AB_</td></tr><tr><td>anti -aTubulin (Mouse)</td><td>DSHE Univ. of Iowa</td><td>Cat#12G, RRID: AB</td></tr><tr><td>IRDye 680RD Donkey anti-rabbit IgG</td><td>LiCOr</td><td>Cat# 9-6, RRID: AB_</td></tr><tr><td>IRDye 800RD Goat anti-rabbit IgG</td><td>LiCOr</td><td>Cat# 9-3, RRID: AB_</td></tr><tr><td>NativePAGE Novex-1% Bis-Tris Protein Gels, 1.0 mm, 1 well</td><td>Life Technologies</td><td>Cat# BN1004BOX</td></tr><tr><td>NativePAGE Novex-1% Bis-Tris Protein Gels, 1.0 mm, 1 well</td><td>Life Technologies</td><td>Cat# BN1002BOX</td></tr><tr><td>Bacterial and virus strains</td><td></td><td></td></tr><tr><td>Rosetta(DE)pLysS Competent Cells</td><td>EMD Millipore (Novagen)</td><td>Cat# 7</td></tr><tr><td>Max Efficiency DH10Bac Cells</td><td>Thermo Fisher Scientific</td><td>Cat# 1-0</td></tr><tr><td>Chemicals, peptides, and recombinant proteins</td><td></td><td></td></tr><tr><td>hTRIC</td><td>This study</td><td>N/A</td></tr><tr><td>hTRIC C1GFP</td><td>This study</td><td>N/A</td></tr><tr><td>hTRIC -tubulin</td><td>This study</td><td>N/A</td></tr><tr><td>hPFD:-tubulin</td><td>This study</td><td>N/A</td></tr><tr><td>hPFD:-tubulin</td><td>This study</td><td>N/A</td></tr><tr><td>TBCA</td><td>This study</td><td>N/A</td></tr><tr><td>TBCB</td><td>This study</td><td>N/A</td></tr><tr><td>Benzonase nuclease</td><td>Sigma-Aldrich</td><td>Cat# E-25KU</td></tr><tr><td>DSS crosslinker H/D</td><td>Creative Molecules</td><td>Cat# 001S</td></tr><tr><td>PMSF</td><td>Sigma-Aldrich</td><td>Cat# P-5G</td></tr><tr><td>CNBr-Activated Sepharose Fast Flow</td><td>GE Healthcare</td><td>Cat# 1</td></tr><tr><td>Apyrase ATPase</td><td>Sigma-Aldrich</td><td>Cat# A-500UN</td></tr><tr><td>GTP</td><td>Thermo Fisher scientific</td><td>Cat# R</td></tr><tr><td>ATP (magnesium salt)</td><td>Sigma-Aldrich</td><td>Cat# A-1G</td></tr><tr><td>Proteinase K</td><td>Sigma-Aldrich</td><td>Cat# P-10Mg</td></tr><tr><td>Sequencing grade Trypsin</td><td>Promega</td><td>Cat# V</td></tr><tr><td>Deposited data</td><td></td><td></td></tr><tr><td>Apo-TRiC map</td><td>This study</td><td>EMDB: EMD-3</td></tr><tr><td>Prefoldin:-tubulin:TRiC complex map</td><td>This study</td><td>EMDB: EMD-3</td></tr><tr><td>Prefoldin:-tubulin:TRiC complex model</td><td>This study</td><td>PDB: 7WU</td></tr><tr><td>TRiC:-tubulin intermediate I model</td><td>This study</td><td>PDB: 7TRG</td></tr><tr><td>TRiC:-tubulin intermediate II model</td><td>This study</td><td>PDB: 7TTN</td></tr><tr><td>TRiC:-tubulin intermediate III model</td><td>This study</td><td>PDB: 7TTT</td></tr><tr><td>TRiC:-tubulin intermediate IV model</td><td>This study</td><td>PDB: 7TUB</td></tr><tr><td>TRiC:-tubulin intermediate I map</td><td>This study</td><td>EMDB: EMD-2</td></tr></table> Continued <table><tr><td>REAGENT or RESOURCE</td><td>SOURCE</td><td>IDENTIFIER</td></tr><tr><td>TRiC:-tubulin intermediate II map</td><td>This study</td><td>EMDB: EMD-2</td></tr><tr><td>TRiC:-tubulin intermediate II map</td><td>This study</td><td>EMDB: EMD-2</td></tr><tr><td>TRiC:-tubulin intermediate IV map</td><td>This study</td><td>EMDB: EMD-2</td></tr><tr><td>Cross-linking mass spectrometry proteomics data</td><td>This study</td><td>PXD</td></tr><tr><td>Human -tubulin</td><td>Adib et al.3</td><td>6</td></tr><tr><td>Human -tubulin</td><td>Alda et al.3</td><td>6</td></tr><tr><td>Human -tubulin</td><td>Aldaz et al.6</td><td>125V</td></tr><tr><td>Tubulin homologue: FtsZ from M. jannaschii</td><td>Lowe et al.4</td><td>1FSZ</td></tr><tr><td>Experimental models: Cell lines</td><td></td><td></td></tr><tr><td>High Five Cells (BTI-TN-5B-4)</td><td>Thermo Fisher scientific</td><td>Cat# B</td></tr><tr><td>Sf cells</td><td>Thermo Fisher scientific</td><td>Cat# 1</td></tr><tr><td>Oligonucleotides</td><td></td><td></td></tr><tr><td>5&#x; pFB--tubulin TTT TAG CCG CCG CCA AAA CAT GAG</td><td>This study</td><td>N/A</td></tr><tr><td>GGA AAT CGT GCA CAT CC</td><td></td><td></td></tr><tr><td>3&#x; pFB--tubulin ATT TTT CTA GAT TAT CAG GCC TCC</td><td>This study</td><td>N/A</td></tr><tr><td>TCT TCG GCC TCC</td><td></td><td></td></tr><tr><td>5&#x; pFB--tubulin TTT TAG CCG CCG CCA AAA CAT GCG</td><td>This study</td><td>N/A</td></tr><tr><td>TGA GTG CAT CTC CAT CC</td><td></td><td></td></tr><tr><td>5&#x; pFB--tubulin ATT TTT CTA GAT TAT CAG TAT TCC</td><td>This study</td><td>N/A</td></tr><tr><td>TCT CCT TCT TCC TCA CC</td><td></td><td></td></tr><tr><td>5&#x; TBCA pST AGT TTC TAG ATT TGT TTA ACT TTA AGA</td><td>This study</td><td>N/A</td></tr><tr><td>AGG AGA TAT ACA TAT GGC CGA TCC TCG CGT GAG</td><td></td><td></td></tr><tr><td>3&#x; TBCA pST AAT TAG GAT CCT CAT TAG TGA TGG</td><td>This study</td><td>N/A</td></tr><tr><td>TGA TGA TGG TGG GCT TCT AAC TTC ACT G</td><td></td><td></td></tr><tr><td>5&#x; TBCB pST AGT TTC TAG ATT TGT TTA ACT TTA AGA</td><td>This study</td><td>N/A</td></tr><tr><td>AGG AGA TAT ACA TAT GGC GGT GAC GGG GGT GTC</td><td></td><td></td></tr><tr><td>3&#x; TBCB pST AAT TAG GAT CCT CAT TAG TGA TGG TGA</td><td>This study</td><td>N/A</td></tr><tr><td>TGA TGG TGA CTC CCT ATC TCG TCC AAC CCG TAG TCC</td><td></td><td></td></tr><tr><td>5&#x; TBCB pFB TAA TTC TCG AGC TAT AAA TAT GGA GGT</td><td>This study</td><td>N/A</td></tr><tr><td>GAC GGG GGT GTC G</td><td></td><td></td></tr><tr><td>3&#x; TBCB pFB TAA TTG GTA CCT CAT TAG TGA TGG TGA</td><td>This study</td><td>N/A</td></tr><tr><td>TGA TGG TGA CTC CC</td><td></td><td></td></tr><tr><td>Recombinant DNA</td><td></td><td></td></tr><tr><td>pFB--tubulin</td><td>This study</td><td>N/A</td></tr><tr><td>pFB--tubulin</td><td>This study</td><td>N/A</td></tr><tr><td>pFB-TRiC</td><td>Knowlton et al.5</td><td>N/A</td></tr><tr><td>pFB-TBCB</td><td>This study</td><td>N/A</td></tr><tr><td>pST-TBCA</td><td>This study</td><td>N/A</td></tr><tr><td>pTS-4 -tubulin template</td><td>Tim Stearns lab</td><td>N/A</td></tr><tr><td>pTS-3 -tubulin template</td><td>Tim Stearns lab</td><td>N/A</td></tr><tr><td>pFB-PFD,2</td><td>Gestaut et al.1</td><td>N/A</td></tr><tr><td>pFB-PFD,5</td><td>Gestaut et al.1</td><td>N/A</td></tr><tr><td>pFB-PFD,6</td><td>Gestaut et al.1</td><td>N/A</td></tr><tr><td>pFB-CCT-GFP,CCT</td><td>Gestaut et al.1</td><td>N/A</td></tr><tr><td>pFB-CCT,CCT</td><td>Gestaut et al.1</td><td>N/A</td></tr><tr><td>pFB-CCT,CCT</td><td>Gestaut et al.1</td><td>N/A</td></tr><tr><td>pFB-CCT,CCT-6XHis</td><td>Gestaut et al.1</td><td>N/A</td></tr><tr><td>Software and algorithms</td><td></td><td></td></tr><tr><td>RELION v.1</td><td>Zivanov</td><td>http://www.mrc-mrb.cam.ac.uk/relion</td></tr><tr><td>PHENIX</td><td>Afonine et al.6</td><td>https://www.phenix-online.org</td></tr></table> (Continued on next page) Continued <table><tr><td>REAGENT or RESOURCE</td><td>SOURCE</td><td>IDENTIFIER</td></tr><tr><td>ChimeraX</td><td>Pettersen et al.7</td><td>https://www.cgl.ucsf.edu/chimera</td></tr><tr><td>ProMod</td><td>Pintilie et al.3</td><td>http://cryoem.bcm.edu/cryoem/downloads/segger</td></tr><tr><td>Igor Pro.22A</td><td>Wavemetrics</td><td>https://www.wavemetrics.com</td></tr><tr><td>Imagequant V.2 and.1</td><td>GE Healthcare Lifesciences</td><td>https://www.gelifesciences.com</td></tr><tr><td>Pymol.8.6.2</td><td>Schrdinger</td><td>https://pymol.org</td></tr><tr><td>CryoSPARC v.2</td><td>Punjani et al.6</td><td>https://cryosparc.com/</td></tr><tr><td>Namdinator</td><td>Kidmose et al.6</td><td>https://namdinator.au.dk/</td></tr><tr><td>T-coffee</td><td>Di Tommaso et al.7</td><td>https://tcoffee.org.eu/</td></tr><tr><td>Mega X</td><td>Hall</td><td>https://www.megasoftware.net/</td></tr><tr><td>Other</td><td></td><td></td></tr><tr><td>Quantifoil R.2/1.3-mesh holey carbon-coated grids</td><td>Quantifoil</td><td>https://www.emsdiasum.com/microscopy/products/grids/quantifoil.aspx</td></tr></table> # RESOURCE AVAILABILITY # Lead contact Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Judith Frydman (ifrydman@stanford.edu). # Materials availability Reagents generated in this study are available from the lead contact upon request without restriction. # Data and code availability The 3D cryoEM density maps have been deposited in the Electron Microscopy DataBank under the accession number EMD: EMD-3, 3, 2, 2, 2, 2. Coordinates have been deposited in the PDB under the accession number PDB: 7WU, 7TRG, 7TTN, 7TTT, 7TUB. The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD. This paper does not report the original code. Any additional information required to reanalyze the data reported in this paper is available from the lead contact upon request. # EXPERIMENTAL MODEL AND SUBJECT DETAILS Initial bacmid DNA was generated in DH10Bac cells (Thermo Fisher Scientific) according to manufacturer protocols. Generation of baculovirus was performed in SF cells (Thermo Fisher Scientific), SF cells were maintained in SF ili SFM at 27C and baculovirus was generated at a cell concentration of $\sim\times$ cells/ml. Protein expression was performed in High Five cells (Thermo Fisher Scientific) adapted to and maintained in ESF media (Expression Systems) at 27C. Viral infections for protein expression were performed at a cell concentration of $\sim\times$ cells/ml. # METHOD DETAILS # Cloning # pFB-$\alpha$ -tubulin and pFB-$\beta$ -tubulin Human $\beta$ -tubulin (TBB) and $\alpha$ -tubulin (TUBA1B) were cloned into the pFB-Dual vector using conventional methods. Briefly, Tubulin was amplified using $5^{\prime}$ primer TTT TAG CGG CCG CCA AAA CAT GAG GGA AAT CGT GCA CAT CC and $3^{\prime}$ primer ATT TTT CTA GAT TAT CAG GCC TCC TCT TCG GCC TCC, and $\alpha$ Tubulin was amplified using $5^{\prime}$ primer TTT TAG CGG CGG CCA AAA CAT GCG TGA GTG CAT CTC CAT CC and $3^{\prime}$ primer ATT TTT CTA GAT TAT CAG TAT TCC TCT CCT TCT TCC TCA CC from the plasmids pTS and pTS respectively. Amplified products and pFB-Dual vector were combined after digestion with NotI and XbaI, and sequence of isolated clones was verified. # TBCA and TBCB TBCA was cloned into pST and TBCB was cloned into pFB-Dual vectors using conventional methods. Briefly, TBCA was amplified using $5^{\prime}$ primer AGT TTC TAG ATT TGT TTA ACT TTA AGA AGG AGA TAT ACA TAT GGC CGA TCC TCG CGT GAG and $3^{\prime}$ primer AAT TAG GAT CCT CAT TAG TGA TGG TGA TGA TGG TGG GCT TCT AAC TTC ACT G and cloned into pST using XbaI and BamHI. TBCB was first amplified using $5^{\prime}$ primer AGT TTC TAG ATT TGT TTA ACT TTA AGA AGG AGA TAT ACA TAT GGA GGT GAC GGG GGT GTC and $3^{\prime}$ primer (with 6XHis) AAT TAG GAT CCT CAT TAG TGA TGG TGA TGA TGG TGA CTG CCT ATC TCG TCC AAC CCG TAG TCC and cloned into pST with Xbal and BamHI. To move TBCB into the pFB-Dual vector for insect cell expression TBCB was PCR amplified out of pST using $5^{\prime}$ TAATTCTCGAGCTATAAATATGGAGGTGACGGGGGTGTCGAND-$3^{\prime}$ TAA TTG GTA CCT CAT TAG TGA TGG TGA TGA TGG TGA CTG CC and inserted into pFB-Dual using Xhol and KpnI. # Protein expression and purification # TRiC alone High Five insect cells (1 L) were co-infected with rBVs encoding his-tagged TRiC with CCT tagged with GFP. Cells were incubated at $2^{\circ}C$ for $\sim 72h$ pelleted by centrifugation at $5\times g$ for $1\min$ , and resuspended in TRiC lysis buffer (1 mM HEPES pH.4, $5~\mathrm{mM}$ NaCl, $2~\mathrm{mM}$ imidazole, $1\%$ glycerol, $5\mathrm{mM}$ PMSF) supplemented with benzonase (Sigma-Aldrich-Aldrich, E) (1,0 units) and a protease inhibitor cocktail [Roche). Cells were lysed using dounce homogenization and debris was cleared by ultracentrifugation at $5,0\times g$ at $4^{\circ}C$ for $4\min$ . Cleared supernatant was passed over nickel resin and washed with column wash buffer ( $5~\mathrm{mM}$ HEPES pH.4, $5~\mathrm{mM}$ NaCl, $5\mathrm{mM}$ $\mathrm{MgCl_}$ $2~\mathrm{mM}$ imidazole, $1\%$ glycerol) with an additional $2~\mathrm{mM}$ NaCl, column wash buffer $+1$ mM ATP, column wash buffer $^+$ an additional $5~\mathrm{mM}$ NaCl, and finally column wash buffer alone. Nickel-bound protein was eluted with elution buffer ( $5~\mathrm{mM}$ HEPES pH.4, $5~\mathrm{mM}$ NaCl, $5\mathrm{mM}$ $\mathrm{MgCl_}$ $4~\mathrm{mM}$ imidazole, $1\%$ glycerol). Protein-containing fractions were pooled and passed over a heparin column equilibrated with MQA buffer ( $5~\mathrm{mM}$ HEPES pH.4, $5~\mathrm{mM}$ NaCl, $5\mathrm{mM}$ $\mathrm{MgCl_}$ $0.5\mathrm{mM}$ EDTA, $1\mathrm{mM}$ DTT, $1\%$ glycerol). Protein was eluted with a linear gradient of $2\% -1\%$ MQB buffer ( $5~\mathrm{mM}$ HEPES pH.4, $1\mathrm{M}$ NaCl, $5\mathrm{mM}$ $\mathrm{MgCl_}$ $0.5\mathrm{mM}$ EDTA, $1\mathrm{mM}$ DTT, $1\%$ glycerol). TRiC-containing fractions were pooled and diluted with MQ buffer ( $5~\mathrm{mM}$ HEPES pH.4, $5\mathrm{mM}$ $\mathrm{MgCl_}$ $1\mathrm{mM}$ DTT, $1\%$ glycerol) to remove excess NaCl. Pooled TRiC was loaded onto a Mono Q ion exchange column and eluted with a $2~\mathrm{mL}$ linear gradient of $0\% -1\%$ MQB. TRiC containing fractions were pooled, concentrated with MQA. TRiC containing fractions were pooled, concentrated, and snap-frozen in liquid nitrogen for long-term storage. # TRiC: $\beta$ -tubulin purification High Five insect cells (1 L) were co-infected with rBVs encoding his-tagged TRiC and the Human $\beta$ -tubulin protein. Cells were incubated at $2^{\circ}C$ for $\sim 72h$ pelleted by centrifugation at $5\times g$ for $1\min$ , and resuspended in TRiC lysis buffer ( $1~\mathrm{mM}$ HEPES pH.4, $5~\mathrm{mM}$ NaCl, $2~\mathrm{mM}$ imidazole, $1\%$ glycerol, $5\mathrm{mM}$ PMSF) supplemented with benzonase (Sigma-Aldrich-Aldrich, E) (1,0 units) and a protease inhibitor cocktail [Roche).Cells were lysed using dounce homogenization and debris was cleared by ultracentrifugation at $5,0\times g$ at $4^{\circ}C$ for $4\min$ . Cleared supernatant was passed over nickel resin and washed with column wash buffer $5~\mathrm{mM}$ HEPES pH.4, $5~\mathrm{mM}$ NaCl, $2~\mathrm{mM}$ imidazole, $1\%$ glycerol). Nickel-bound protein was eluted with column wash buffer containing $4~\mathrm{mM}$ imidazole. Protein-containing fractions were pooled and passed over a heparin column equilibrated with MQA buffer ( $5~\mathrm{mM}$ HEPES pH.4, $5~\mathrm{mM}$ NaCl, $5\mathrm{mM}$ $\mathrm{MgCl_}$ $0.5\mathrm{mM}$ EDTA, $1\mathrm{mM}$ DT, $1\%$ glycerol). Protein was eluted with a linear gradient of $2\% -1\%$ MQB buffer ( $5~\mathrm{mM}$ HEPES pH.4, $1\mathrm{M}$ NaCl, $5\mathrm{mM}$ $\mathrm{MgCl_}$ $0.5\mathrm{mM}$ EDTA, $1\mathrm{mM}$ DTT, $1\%$ glycerol). TRiCcontaining fractions were pooled and diluted with MQ buffer ( $5~\mathrm{mM}$ HEPES pH.4, $5\mathrm{mM}$ $\mathrm{MgCl_}$ $1\mathrm{mM}$ DTT, $1\%$ glycerol) to remove excess NaCl. Pooled TRiC was loaded onto a Mono Q ion exchange column and eluted with a $20O~\mathrm{mL}$ linear gradient of $0\% -1\%$ MQB. Fractions were assayed for the presence of $\beta$ -tubulin and TRiC, pooled, concentrated with a $1\mathrm{kDa}$ MWCO Centricon device, and passed over a Superose size exclusion column equilibrated with MQA. TRiC containing fractions were pooled, concentrated, and snap-frozen in liquid nitrogen for long-term storage. The integrity of TRiC and $\beta$ -tubulin occupancy were confirmed by SDS-PAGE, native-PAGE, Coomassie staining, and immunoblotting with a rabbit CCT-specific antibody (Abcam ab) and a mouse $\beta$ -tubulin specific monoclonal antibody (Proteintech-1-Ig). # Human Prefoldin All Human Prefoldin subunits were co-expressed using baculovirus in High Five insect cells. The Prefoldin complex was isolated by affinity (Nickel-NTA), anion exchange (Mono Q/1), and gel filtration (Superdex) chromatography. At each step, fractions containing Prefoldin were identified by SDS-PAGE. Final protein was concentrated to $\sim~\mu M$ using an Amicon Ultra kDa MWCO, and $5\%$ glycerol was added to yield a final concentration of $1\%$ . Protein was aliquoted, snap-frozen, and stored at $-8^{\circ}C$ . # Human Prefoldin + $\alpha$ Tubulin, $\beta$ Tubulin Human Prefoldin was co-expressed with human $\alpha$ -tubulin or $\beta$ -tubulin in High Five insect cells and purified as described above without the anion exchange step. All subunits were co-expressed using baculovirus in High Five insect cells. The Prefoldin complex was isolated by affinity (NickelNTA) and gel filtration (Superdex) chromatography. At each step, fractions containing Prefoldin and Tubulin were identified by SDS-PAGE. Final protein was concentrated to $\sim~\mu M$ using an Amicon Ultra kDa MWCO, and $5\%$ glycerol was added to yield a final concentration of $1\%$ . Protein was aliquoted, snap-frozen, and stored at $-8^{\circ}C$ . # TBCA TBCA was expressed in L of BL Rosetta pLysS cells O/N at $1^{\circ}C$ . Cells were lysed in $5~\mathrm{mL}$ of column buffer (3 mM NaCl, $5~\mathrm{mM}$ HEPES pH.4) with benzonase, fresh PMSF, $1~\mathrm{mM}$ imidazole and Complete protease inhibitors, EDTA free. Cells were lysed 2X using an emulsiflex (Avestin), and lysate was cleared at $2,0\times g$ for $3~\mathrm{min}$ . Protein was affinity purified in a $1~\mathrm{mL}$ Ni column equilibrated with column buffer and eluted with column buffer $+4~\mathrm{mM}$ imidazole after a wash of $\sim~\mathrm{mL}$ column buffer $+1~\mathrm{mM}$ imidazole. Protein was concentrated using an Amicon ultra kD MWCO device to less than mL and run over an Superdex SEC column equilibrated with column buffer $+1$ mM DTT. Protein was concentrated using an Amicon ultra kD MWCO device, glycerol added to $5\%$ , aliquoted, snap-frozen, and stored at $-8^{\circ}\mathrm{C}$ . # TBCB TBCB was expressed in Hi-5 cells adapted to ESF media by infection with $1\mathrm{mL}$ of P virus into L of cells at $-2\times^{6}$ cells/ml. Cells were harvested days after infection. Cells were lysed in column buffer (3 mM NaCl, 5 mM HEPES pH.4) with Benzonase, fresh PMSF, $2\mathrm{mM}$ imidazole, and Complete protease inhibitors, EDTA free. Cell pellet was resuspended in $\sim\mathrm{mL}$ lysis buffer and lysed using dounce homogenization. Lysate was cleared at $4,0\times$ for $4\mathrm{min}$ . Protein was affinity purified in a $2\mathrm{mL}$ Ni column equilibrated with column buffer and eluted with column buffer $+4\mathrm{mM}$ imidazole after washes of $\sim\mathrm{mL}$ column buffer $+1\mathrm{mM}$ imidazole, $5\mathrm{mL}$ column buffer $+5\mathrm{mM}$ NaCl $+1\mathrm{mM}$ imidazole, $5\mathrm{mL}$ column buffer $+1\mathrm{mM}$ imidazole. Protein was concentrated using an Amicon ultra kD MWCO to less than $2\mathrm{mL}$ and run over an Superdex SEC column equilibrated with $2\mathrm{mM}$ NaCl, $5\mathrm{mM}$ HEPES pH.4, $1\mathrm{mM}$ TCEP. Protein was concentrated using an Amicon ultra kD MWCO device, glycerol added to $5\%$ , aliquoted, snap-frozen, and stored at $-8^{\circ}\mathrm{C}$ . # Native and SDS PAGE analysis # Native and SDS gel analysis for prefoldin-tubulin Prefoldin samples were diluted to $5\mu M$ For native gels $7.5\mu L$ of sample was run out on $4 -1\%$ Native-PAGE gels. For SDS-PAGE, $8\mu L$ of 4XPSB $2\mathrm{mM}$ Tris pH.8, $8\%$ SDS, $4\%$ glycerol, $2\%$ $\beta$ -mercaptoethanol, bromophenol blue to color) was added to $2\mu L$ of sample, and $1\mu L$ of each sample was separated in $1\%$ SDS-PAGE gels. Both gels were run in duplicate with one used for Coomassie staining and the other for immunoblot. # Native and SDS gel analysis for TRiC:tubulin TRiC samples were diluted to $0.5\mu g / \mu L$ and bovine Tubulin to $\sim.0\mu g / \mu L$ for each Tubulin. For native gels, $5\mu L$ of TRiC samples was separated. SDS-PAGE samples were made by adding 4XPSB to $1\times$ final concentration, $6\mu L$ of TRiC samples and, 2, 4 and $8\mu L$ of Tubulin were separated. # Tubulin folding assays $\beta$ -tubulin folding assay was performed in folding buffer (2 mM Tris-HCl pH.4, $5~\mathrm{mM}$ KCl, $1\mathrm{mM}$ $\mathrm{MgCl_}$ and $0.2\mathrm{mM}\mathrm{CaCl_}$ with fresh TCEP added to $1\mathrm{mM}$ . Prefoldin: $\beta$ -tubulin was prepared to a final concentration of $1\mu M$ ,TRiC to $0.1\mu M$ and TBCA to $1\mu M$ Samples were incubated in the presence/absence of $0.9\mathrm{mM}$ ATP, 1 mM GTP as indicated for h. Folding assay samples were separated by native PAGE and $\beta$ -tubulin was detected by immunoblot with a monoclonal mouse antibody (Proteintech,2-1-Ig). $\alpha$ -tubulin folding was performed in folding buffer with an additional $5~\mathrm{mM}$ KCl and fresh TCEP added to $1\mathrm{mM}$ . Prefoldin: $\alpha$ -tubulin was prepared to a final concentration of $1\mu M$ ,TRiC to $0.2\mu M$ and TBCB to $1\mu M$ . Samples were incubated in the presence/absence of $0.9\mathrm{mM}$ ATP, $1\mathrm{mM}$ GTP as indicated for $1h$ at 37C. Folding assay samples were separated by native PAGE and $\alpha$ -tubulin was detected by immunoblot with a monoclonal mouse antibody (Sigma-Aldrich T). # Proteinase K assays Prefoldin:tubulin, Bovine tubulin, TRiC, and mmCPN were brought to a final concentration of $1\mu M$ in 1XATPase buffer $(5~\mathrm{mM}$ HEPES, $5~\mathrm{mM}$ KCl, $5\mathrm{mM}\mathrm{MgCl_}$ $1\%$ glycerol, $1\mathrm{mM}$ DTT). Samples were brought to $3^{\circ}C$ and $1\mathrm{mM}$ ATP/GTP was added to the Tubulin dimer control, Prefoldin:tubulin sample, TRiC:Prefoldin:tubulin sample $(+\mathsf{ATP / GTP})$ , and TRiC:Prefoldin:tubulin sample $(+\mathsf{ATP / GTP -AIFx})$ . After min, NaF to $1\mathrm{mM}$ and $A1N O_{3}$ to $6\mathrm{mM}$ was added to the TRiC:Prefoldin:tubulin sample $(+\mathsf{ATP / GTP -}$ AIFx). After $3\min$ samples were brought to room temperature. $1\mu L$ was removed from each sample for zero-minute time points, and digestion was initiated by addition of proteinase K to a final concentration of $2ng / \mu L$ . Timepoints were taken at, 4, 8 and $1\min$ by adding $1\mu L$ sample to $2\mu L$ of PMSF, followed by $6\mu L$ 4XPSB. Samples were analyzed by SDS-PAGE followed by immunoblot with indicated antibodies. For assays with denatured Tubulin, chaperones were diluted to $0.2\mu M$ . Bovine Tubulin at a concentration of $1.5\mu M$ was denatured in $6M$ GuHCl, $1~\mathrm{mM}$ HEPES pH.4, $1\mathrm{mM}$ DTT, and chaperone mixes were used to dilute denatured Tubulin:1 for a final concentration of $0.1\mu M$ . Aggregated Tubulin was pelleted by centrifugation at $21k^{\star}G$ for $1\min$ . Supernatant was removed and PK assay was carried out as described as above with $1\mu L$ timepoints. Tubulin binding by chaperones was assayed by running $1\mu L$ samples in Native PAGE. For western blots, $\beta$ -tubulin was probed for with a monoclonal mouse antibody toward the N-terminus (Proteintech,2-1-lg), a polyclonal rabbit antibody (abcam ab) or a C-terminal epitope specific AA antibody (Millipore sigma-6), and $\alpha$ -tubulin with a mouse monoclonal antibody (Sigma-Aldrich T). # Crosslinking-mass spectrometry For hPrefoldin: $\beta$ -tubulin with hTRiC samples, TRiC was brought to $1\mu M$ and Prefoldin to $2\mu M$ hPrefoldin: $\alpha /\beta$ -tubulin alone was prepared at $5\mu M$ ,hTRiC: $\beta$ -tubulin alone was prepared at $1\mu M$ . All samples were prepared in $5~\mathrm{mM}$ HEPES pH.4, $5~\mathrm{mM}$ NaCl and $1\mathrm{mM}$ DTT. TRiC samples were incubated at $3^{\circ}C$ for $3\min$ with nucleotide to reach equilibrium/closed state. Fresh DSS crosslinker dissolved in DMsO was added to a final concentration of $1\mathrm{mM}$ , TRiC samples were incubated at $3^{\circ}C$ for $1h$ with crosslinker, hPrefoldin: $\alpha /\beta$ -tubulin alone was incubated at room temperature for $1h$ at room temperature. Crosslinking was quenched by incubation for $3\min$ after addition of $1M$ Tris pH.4 to a final concentration of $1~\mathrm{mM}$ . Crosslinked samples processing steps included reduction and alkylation of cysteine residues with tris(2-carboxyethyl)phosphine and iodoacetamide, respectively, sequential digestion with endoprotease Lys-C (Wako) and trypsin (Promega), clean-up using solid-phase extraction (Waters Sep-Pak tC cartridges) and fractionation of the purified digests by size exclusion chromatography (SEC; GE Superdex Peptide PC $3.2 / 3$ ).<sup>6</sup> Liquid chromatography-tandem mass spectrometry (LC-MS/MS) was performed on an Easy nLC-1 HPLC system coupled to an Orbitrap Fusion Lumos mass spectrometer (both ThermoFisher Scientific). Peptides were separated by reversed-phase chromatography on an Acclaim PepMap RLSC C column (2 mm x m, ThermoFisher Scientific) at a flow rate of nL/min. The mobile phase gradient was4% B in min, with mobile phases A = water/acetonitrile/formic acid (9:2:0.1, v/v/v) and B = acetonitrile/water/formic acid (8:2:0.1, v/v/v). MS/MS data were acquired in the data-dependent acquisition top speed mode with a cycle time of s. Precursor and fragment ion spectra were acquired in the Orbitrap at,0 and,0 resolution, respectively. Precursor ions with a charge state of +3 to +7 were fragmented in the linear ion trap at a normalized collision energy of%. Dynamic exclusion was enabled for s after one sequencing event. MS/MS spectra were analyzed using xQuest, version.1.5 (available from https://gitlab.ethz.ch/leitner_lab/xquest_xprophet). The database contained the entries of all human TRiC and prefoldin subunits, human tubulin beta (TBB5_HUMAN), and insect tubulins as possible contaminants. No other contaminant proteins were observed at relevant levels. Search parameters for xQuest included: enzyme = trypsin, maximum number of missed cleavages = 2, carbamidomethylation of Cys as fixed modification, oxidation of Met as variable modification, Lys and protein N terminus as cross-linking sites, mass error tolerances of 1 ppm at the MS level and 2 ppm at the MS level. Search results were further filtered with stricter mass tolerances depending on the dataset, and identifications were required to have an xQuest delta score <0.9, a TIC score >0.1, and a minimum of four bond cleavages overall or three consecutive bond cleavages per peptide. A parallel search against the reversed and shuffled sequences of the database entries revealed no decoy hits fulfilling the search and filter criteria, suggesting the false discovery rate is close to% for the selected score thresholds. XLs are shown in Table S. Mapping of XLs was performed using xiVIEW.<sup>6</sup> # Native Mass spectrometry Prefoldin and Prefoldin: $\beta$ -tubulin were diluted-fold from storage buffer into mM ammonium acetate, pH.0, before completing buffer exchange using xDa MWCO Amicon centrifugal filters (Millipore Sigma). Final concentrations were estimated by UV absorbance to be between and $\mu$ M Prefoldin. Native mass spectra were then recorded within several hours, during which proteins were kept on ice. Samples were injected using borseilicate emitters (Thermo Fisher Scientific E) into a QExactive Plus EMR orbitrap mass spectrometer (ThermoFisher Scientific). The quadrupole was set to transmit m/z range,0 to,0. Resolution was,5 at m/z for a transient time of ms. Injection time was ms and automatic gain control was off. The capillary voltage was.2 kV and S-lens RF was. Microscans were grouped sequentially in sets of. In-source activation was V and activation in the higher-energy collisional dissociation (HCD) cell was V, based on values found to sharpen peaks without discernable complex dissociation. Pressure in the HCD cell (nitrogen) was 5e-1 mbar. For transmission of Prefoldin, the injection flatapole was set to V, the inter flatapole lens to V, and the bent flatapole to V; for Prefoldin: $\beta$ -tubulin transmission, these values were respectively, 1, and V. Raw data were summed and visualized in XCalibur v.1.3.9 (ThermoFisher Scientific). Initial deconvolution was performed using UniDec v.1.0. <sup>7</sup> Masses were measured manually by minimizing error over different charge state assignments. Each charge state arising from Prefoldin and Prefoldin: $\beta$ -tubulin was assigned to the open or compact conformation based on Gaussian peak envelopes. Raw signal intensities of each peak were recorded from XCalibur, summed within assigned conformations, and used to derive fractional occupancies. Conclusions based on native MS data were cross-validated by computing changes in solvent-accessible surface area for Prefoldin models. # Cryo-EM specimen preparation and data collection Apo-TRiC $\mu$ L of apo-TRiC sample was vitrified at mg/mL and applied to-mesh R.2/1.3 holey-carbon grids (Quantifoil) coated with PolyL-lysine and vitrified using Vitrobot Mark IV (Thermo Fisher Scientific, CMCI in Seoul). 6,0 Movies were collected on a Titan Krios (Thermo Fisher Scientific) equipped with a Falcon (Thermo Fisher Scientific) detector. The detailed imaging conditions are shown in Table S. # Prefoldin: $\beta$ -tubulin:TRiC.5 $\mu$ M TRiC sample was mixed with $\mu$ M Prefoldin: $\beta$ -tubulin complex at RT. CryoEM grids (Quantifoil R.2/1.3 Cu) were glow discharged (PELCO easiGlow) for s.0% Octyl-beta-glucoside was mixed with the sample prior to vitrification. Sample was vitrified using Vitrobot Mark IV, where.7 $\mu$ L sample was applied on the cryoEM grids and blotted for s,7 movies were collected on a Titan Krios (Thermo Fisher Scientific) equipped with the K Summit (Gatan) detector. The detailed imaging conditions are shown in Table S. # TRiC: $\beta$ -tubulin with ATP-AIFx $\mu$ M copurified TRiC: $\beta$ -tubulin sample was incubated with ATP-AIFx for h atC. CryoEM grids (Quantifoil R.2/1.3 Cu) were glow discharged (PELCO easiGlow) for s. Sample was vitrified using a Gatan Leica GP plunger, where.7 $\mu$ L sample was applied on the cryoEM grids and blotted from the back for5 s,2 movies were collected on Titan Krios (Thermo Fisher Scientific) equipped with the K Summit (Gatan) detector. The detailed imaging conditions are shown in Table S. # Data processing and 3D refinement All image processing was done in RELION $3.1^{6}$ and cryoSPARC v.2.6 Computing resources were utilized in the $S^{2}C^{2}$ SLAC national facility and at CMCI at Seoul National University. The detailed processing parameters are summarized in Table S. # Apo-TRiC Movies were aligned in $5\times$ patches in MotionCor and CTF parameters were estimated with GCTF.6 Utilizing template-based autopicking in cryoSPARC v.2, 2,1,2 particles were initially picked. After 2D classification and removing bad particles, 9,2 particles were subjected to 3D heterogeneous refinement using Ab initio model in cryoSPARC. After further 3D classification and CTF refinements, non-uniform refinement was performed using,7 particles yielding a $3.1\mathring{A}$ map of apo-TRiC based on the gold-standard Fourier shell correlation (ESC) at.1. This analysis workflow is illustrated in Figure S. # Prefoldin: $\beta$ -tubulin:TRiC Movies were aligned in $5\times$ patches in MotionCor, and CTF parameters were estimated with GCTF. After initial template-based picking of,6,6 particles, 2D classification was performed. After selection, 4,8 particles were used for 3D classification yielding major classes. Prefoldin bound TRiC with,0 particles and non-bound TRiC with,8 particles. Using non-uniform refinement, prefoldin bound and non-bound TRiC was reconstructed at a resolution of $3.9\mathring{A}$ and $4.2\mathring{A}$ respectively. This analysis workflow is illustrated in Figure S. # TRiC: $\beta$ -tubulin with ATP-AIFx,4 movies were aligned in $5\times$ patches by MotionCor (v), and CTF parameters were estimated with CTFFIND (v.1). Particles picked with a template matching method were subject to multiple rounds of 2D classification followed by 3D classification. 8,5 particles in open state were used to reconstruct an open state TRiC: $\beta$ -tubulin map to $3.8\mathring{A}$ and,1 particles were used to obtain a $2.7\mathring{A}$ C symmetry map. This analysis workflow is illustrated in Figure S4A. # Compositional heterogeneity analysis of TRiC: $\beta$ -tubulin with ATP-AIFx,1 particles of closed state are aligned to the C (D) symmetry axis to obtain a $2.5\mathring{A}$ consensus map of closed state TRiC-tubulin. For compositional analysis of tubulin in the TRiC chamber, the star file of the related particles was extended times by religion_particle_symmetry_expand with D symmetry to cover the orientation for each of two rings in each particle. 3D focused classification of the tubulins was performed with a tubulin mask but without symmetry and orientational search. The identified tubulin occupied rings were thus traced back to the original double-ring TRiC particles for the tubulin occupancy analysis. This analysis workflow is illustrated in Figure S4A. # Conformational heterogeneity analysis of TRiC: $\beta$ -tubulin with ATP-AIFx The tubulin occupied rings were further subject to focused classification for conformational heterogeneity analysis. In brief, a tubulin mask was applied to the related rings in multiple 3D classifications without orientational search, four tubulin intermediate conformations were identified and each conformation was further locally refined to high resolution. This analysis workflow is illustrated in Figure S4B. # Model building # Prefoldin: $\beta$ -tubulin:TRiC Model building started from previous Prefoldin:TRiC model (PDB: 6NR).1 The initial model was fitted into the density map and manually refined in COOT.6 Then, the model was refined on the Namdinator server using MDFF and Phenix real space refinement default options.6 After few rounds, the model was further corrected using Phenix and COOT.6 # Copurified TRiC: $\beta$ -tubulin under ATP-AIFx condition The model of TRiC in the closed form (PDB: 7LUM) reference was rigidly fit into the closed state TRiC density by rigid body fitting with Fit in Map tool from Chimera v.1. This fitted model was further refined with phenix.real_space refine in Phenix v.1.1, ISOLDE v.1.0. And refined models were inspected and adjusted in COOT. The model of tubulin (PDB: 6l2l)6 was rigidly fitted to each tubulin intermediate density, and manually adjusted by COOT and ISOLDE v.1.0. All adjusted models were then refined using phenix.real_space_refine in Phenix. All models were validated by Q-score and phenix. validation, cryoem.6 Difference maps showing the nucleotide density in TRiCtubulin closed state were calculated between the complex density map and the map calculated from the model of protein only, generated by phenix. real_diff map. The figures of the difference map were generated by Chimera with the same contour level sigma. All other figures were generated by Chimera and ChimeraX.7 # PISA analysis The interactions between TRiC and $\beta$ -tubulin model in each state were calculated using the PISA server (https://www.ebi.ac.uk/ pdbe/pisa/). # Logo plot generation Residue conservation was compared across mainly eukaryotic with some archaeal species (group II chaperonins) for residues predicted to make interactions by PISA. Plots were generated using EDlogo plot.7 # Sequence alignment and phylogenetic tree building Multiple sequence alignments for tubulin and tubulin homologues (Data S) were generated using the multiple sequence alignment tool T-coffee Espresso and visualized. $^{7}$ A phylogenetic tree of tubulin and tubulin homologs was made in MEGA X software. $^{7}$ # QUANTIFICATION AND STATISTICAL ANALYSIS Statistical tests were carried out in Prism. Pie, bar charts, heat maps and scatterplots were produced using R and Prism. Gel and radiograph densitometry were done in ImageJ. ATPase data was plotted in Prism. Average XL distance for PFD-TRiC interXLs was calculated by mapping all interXLs in which both XL residues are resolved onto PDB: 6NR and averaging the distance. SEM was calculated by dividing the SD by the square root of the number of XLs. # Supplemental figures  # Figure S. Purification systems and folding analysis for $\alpha /\beta$ -tubulin, related to Figure (A) Purification scheme for Prefoldin:tubulin and TRiC: $\beta$ -tubulin (B) Purified TRiC: $\beta$ -tubulin or $\alpha$ -tubulin with SDS-PAGE and Native-PAGE analysis, immunoblot confirms $\beta$ -tubulin or $\alpha$ -tubulin comigrates with TRiC in native gel. (C) Purified Prefoldin:tubulin ( $\beta$ or $\alpha$ -tubulin) with SDS-PAGE and Native-PAGE analysis, note faster migration of Prefoldin:tubulin relative to Prefoldin alone in native gel. (D) $\alpha$ -tubulin folding assay. $\alpha$ -tubulin co-expressed and copurified with Prefoldin comigrates on a native gel. In the absence of nucleotide $\alpha$ -tubulin comigrates with TRiC, and addition of nucleotide (ATP/GTP) drives folding and partial release of $\alpha$ -tubulin in the presence of TBCB. (E) Protease sensitivity assay (Proteinase K) of Prefoldin:tubulin samples; Fully folded $\alpha /\beta$ Tubulin dimer is insensitive to PK digestion, while Prefoldin:tubulin samples show rapid degradation by SDS-PAGE visualized by Coomassie stain (i) or immunoblot (ii). (F) Protease sensitivity assay of Prefoldin: $\alpha$ -tubulin folding by TRiC. Like $\beta$ -tubulin, $\alpha$ -tubulin bound to Prefoldin is protease sensitive, transfer to TRiC from Prefoldin causes a decrease in sensitivity, and addition of nucleotide drives folding resulting in limited protease sensitivity equivalent to $\alpha$ -tubulin in a native $\alpha /\beta$ Tubulin dimer. (G-1) Protease sensitivity assay to test if TRiC closure is providing protease protection with degradation assayed by both an N-terminal region polyclonal antibody, and a C-terminal (AA) monoclonal antibody. After completion of $\beta$ -tubulin folding by TRiC as assayed in (H), apyrase is added to reopen the lid TRiC complex, followed by incubation at the indicated times. PK protease sensitivity is assayed at indicated time points (I). No increase in protease sensitivity is observed immediately after min apyrase addition, the min incubation leads to a mild increase in sensitivity likely due to slow unfolding.  # Figure S. Native MS and XL analysis for prefoldin and prefoldin:-tubulin, related to Figure (A) (Scheme of XL-MS analysis pipeline. XL of Prefoldin:tubulin (i), Prefoldin:tubulin:TRiC (i), and tubulin:TRiC (ii) samples were carried out in the indicated nucleotide condition and analyzed by Native PAGE and SDS-PAGE as indicated, prior to Mass Spectrometry analysis. Native PAGE analysis shows XLed samples run as single complexes, while SDS-PAGE shows XLed samples migrate as high molecular weight multi-protein species due to inter-subunit Xling. (B) Bottom view of XLs mapped onto Prefoldin, note extensive XLs between PFD and PFD. (C) Representative native mass spectrum of Prefoldin without $\beta$ -tubulin. Two charge state distributions can be inferred to correspond to conformations that are relatively open and compact, represented by cartoons with more or fewer charges (+), the calculated solvent exposed surface area is listed next to the cartoons for open (PDB:6NRB) $^{1}$ and compact (PDB:6NRB) $^{1}$ Prefoldin. The most abundant charge state of each conformation is labeled. (D) Mass to charge plot of deconvolved data in (C). A single mass, Prefoldin, gives rise to both peak distributions of charge. (E) Alternative representation of the native mass spectrometry data in , with charge plotted on the vertical axis and intensity as a color scale (right). Open and compact states of Prefoldin and Prefoldin: $\beta$ -tubulin are outlined in blue and red respectively, with outline widths proportional to relative abundances. These data show a shift in conformational equilibrium upon $\beta$ -tubulin binding. (F, G) Prefoldin structures alone (i) and with the top ring of TRiC (ii) from the least engaged (II) (PDB:6NRB) $^{1}$ and most engaged (G) (PDB:6NRB) $^{1}$ Prefoldin:TRiC structures. The distance between the terminally resolved residues of PFD (Ile) and PFD (Glu) are indicated in magenta.  # Figure S. Cryo-EM workflow and XL analysis for prefoldin: $\beta$ -tubulin-TRiC, related to Figure (A) Representative cryoEM micrograph and 2D averages of apo-TRiC and the complex of Prefoldin: $\beta$ -tubulin-TRiC in the open conformation. (B) Image processing workflow of apo-TRiC and Prefoldin: $\beta$ -tubulin-TRiC and Gold standard FSC curve (C). (D) Local resolution estimation of three reconstructed maps: apo-TRiC, TRiC: $\beta$ -tubulin (Prefoldin unbound), Prefoldin:TRiC: $\beta$ -tubulin. (E) Crosslink mapping on Prefoldin: $\beta$ -tubulin bound to TRiC (F) Histogram of shortest measured intra/inter-ring $\mathrm{C}_{\alpha}$ -$\mathrm{C}_{\alpha}$ distances for identified XLs between Prefoldin and TRiC.  # Figure S. Cryo-EM workflow and analysis for TRiC: $\beta$ -tubulin, related to Figure (A) Representative micrograph and 2D class averages of TRiC: $\beta$ -tubulin with ATP-AlFx. (B) Processing workflow of TRiC: $\beta$ -tubulin with ATP-AlFx and (C) 3D classification strategy for focusing the density inner chamber of TRiC. Our compositional heterogeneity analysis identifies different $\beta$ -tubulin occupancies for TRiC, including no substrate bound (apo state), and single or double ring occupancy. Further analysis identified four progressively folded tubulin conformations inside the chamber: State I (4% of the rings); State II (1% of the rings); State III (3% of the rings) and State IV (9% of the rings) corresponding to folded tubulin. (D) Local resolution map estimation of $\beta$ -tubulin intermediate states. (E) FSC curve of reconstructed maps showing resolution according to the gold standard FSC (0.1 criterion). (F) Q-score plots of the $\beta$ -tubulin intermediate states shown with the expected Q-score according to.5 and.0 . (G) C-alpha backbone RMSD per residue between intermediate state and native $\beta$ -tubulin (PDB:6l2l). Large variation indicated at T loop (red star) and N loop (yellow hexagon).  # Figure S. TRiC chamber spatially orients and restrains $\beta$ -tubulin, related to Figure (A) CryoEM density of $\beta$ -tubulin E hook tail in the CCT/4 pocket for the different folding states (I-IV). (B) cryoEM density of CCT C terminal tail connected to $\beta$ -tubulin for the different folding states (I-IV). (C) cryoEM density of the $\beta$ -tubulin at the GTP binding pocket: T loop (red star), N loop (yellow hexagon), nucleotide (magenta eclipse). (D) Difference map analysis of the nucleotide occupancy in $\beta$ -tubulin GTP binding pocket in comparison to CCT ATP binding pockets. The difference maps are displayed with $+3\sigma$ . (E) Slice side and top view of CCT/2C terminal tail contacts in apo closed TRiC. (F) Electrostatic surface display of human $\alpha$ -tubulin (PDB:6I2I), $\gamma$ -tubulin (PDB: this study) and $\gamma$ -tubulin (1Z5V1C), respectively. (G) Logo plot of the conservation of CCT residues. The residues making direct contact with $\beta$ -tubulin are indicated by red arrows.  # Figure S. Tubulin coevolved with TRiC for directed folding process, related to Figure (A) Sequence alignment for selected areas of human $\alpha$ , $\beta$ , $\gamma$ -tubulins and archaeal/prokaryotic homologues (CetZ: H. Volcanii CetZ: H. Volcanii, FtsZ: M. jannaschii and TubZ: B.careus). T loop, GTP binding motif, N/C-domain interface and C-domain including H and E-hook are represented. Red box indicates conserved N and L in T loop and N/C-domain interface in human tubulins. (B) Phylogenetic tree of tubulins and their structures. Structures are colored in domain (N-domain: cyan, C domain: green, Core domain: blue, M-domain: red). The scale bar represents the number of amino acids substitutions per site. (C) The hydrophobic core and domain interfaces of $\beta$ -tubulin and FtsZ with hydrophobic residues (Trp, Phe, Ile, Leu) highlighted with yellow ( $\beta$ -tubulin) and gray (FtsZ) balls. Red arrow indicates the N/C-domain interface. Blue arrows and dashed line indicate the C/Core-domain interface. Colored circles represent hydrophobic core of each domain. The progressive assembly of the hydrophobic cores for N-, C-and Core domain of $\beta$ -tubulin (bottom panels). (D) Model B factor distribution of each state highlights the N-domain with T loop in red dashed box (i) and corresponding cryo-EM density maps with fitted model are presented (ii). Per residue RMSD plots of T loop among four states (iii) and Q-score plot of T loop (iv). Dashed black line indicates average Q score expected at.5  resolution. (E) $\beta$ -tubulin model with point mutations at the GTP binding pocket indicated as balls on residues that have folding defects. Their locations and effects are summarized. # Update # Cell Volume, Issue, 2 April, Page DOI: https://doi.org/1.1/j.cell.2.0.0 Correction # Structural visualization of the tubulin folding pathway directed by human chaperonin TRiC/CCT Daniel Gestaut, Yanyan Zhao, Junsun Park, Boxue Ma, Alexander Leitner, Miranda Collier, Grigore Pintilie, Soung-Hun Roh,\* Wah Chiu,\* and Judith Frydman\* *Correspondence: sbrob@enu.ac.kr (S. H.R.), wobe@stanford.edu (W.C.), ifyrdman@stanford.edu (J.F.) https://doi.org/1.1/j.cell.2.0.0 (Cell, 4-4. e-e; December, 2) We overlooked a related study that was published during the revision of the manuscript. This study visualized tubulin within immu. noprecipitated TRiC complexes using cryo-EM and found partial tubulin density within the TRiC chamber similar to what we observe in  (Kelly et al, 2). We have now acknowledged this study in the Discussion and have cited it accordingly. We apologize for any confusion this error may have caused.